<meta http-equiv="Content-Type" content="text/html; charset=utf-8">  <meta name="node-id" content="2809"> <meta name="revision-id" content="16539"> <meta name="cms-base-url" content="http://cms.ncbi.nlm.nih.gov"> <meta name="cms-view-url" content="http://cms.ncbi.nlm.nih.gov/genbank/dbgss"> <meta name="cms-edit-url" content="http://cms.ncbi.nlm.nih.gov/node/2809/edit"> <meta name="created" content="2013-04-19T13:18:37-04:00"> <meta name="modified" content="2013-08-01T10:24:07-04:00"> <meta name="publication-date" content="2013-04-19T13:13:00-04:00"> <meta name="author" content="yankie"> <meta name="subsite" content="genbank"> <meta name="path" content="genbank/dbgss"> <meta name="node-type" content="page"> <meta name="jira-ticket" content=""> <meta name="cms-tags" content="">  <meta name="" content=""> <title>GSS</title>

<div class="node clear-block">

<div class="content">

## What is GSS?

The GSS division of GenBank is similar to the EST division, with the exception that most of the sequences are genomic in origin, rather than cDNA (mRNA). It should be noted that two classes (exon trapped products and gene trapped products) may be derived via a cDNA intermediate. Care should be taken when analyzing sequences from either of these classes, as a splicing event could have occurred and the sequence represented in the record may be interrupted when compared to genomic sequence. The GSS division contains (but is not limited to) the following types of data:

*   random "single pass read" genome survey sequences.
*   cosmid/BAC/YAC end sequences
*   exon trapped genomic sequences
*   Alu PCR sequences
*   transposon-tagged sequences

Section 1.3.3 of the GenBank 96.0 [release notes](ftp://ftp.ncbi.nih.gov/genbank/release.notes/gb96.release.notes) provides additional information about the GSS division.

Although dbGSS sequences are incorporated into the GSS Division of [GenBank](/~/genbank), annotation in dbGSS is more comprehensive and includes detailed information about the contributors, experimental conditions, and genetic map locations.

## Other ways to access dbGSS

Other ways [to access](/~/dbGSS/access.html) dbGSS.

## How to submit data

How to [submit data](/~/dbGSS/how_to_submit.html) to dbGSS.

## dbGSS summary by organism

Information on the [current release](/~/dbGSS/dbGSS_summary.html).

</div>

</div>