

<html>
  <head>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta charset="UTF-8" />
    <title></title>
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <title>GenBank Submission Tools</title>
  </head>

  <body>

    
      <div class='node clear-block'>
        <div class='content'>

          
            <h1 id="submitting-sequences-to-dbest">Submitting sequences to dbEST</h1>
<p>Please ensure that your submissions follow these sequence quality and file formatting guidelines:</p>
<h2 id="sequence-quality-and-content">Sequence Quality and Content</h2>
<ol>
<li><p>Please submit only single-pass sequences (no next-gen sequences, raw or assembled)</p>
</li>
<li><p>Remove: vector, linker, adaptor, mitochondrial, ribosomal, and contaminant sequences</p>
</li>
<li><p>Use VecScreen (<a href="http://www.ncbi.nlm.nih.gov/tools/vecscreen/" target="_blank">http://www.ncbi.nlm.nih.gov/tools/vecscreen/</a>) to identify any vector, linker, or adaptors (accepts multiple sequences in fasta format)</p>
</li>
<li><p>. Trim terminal N&apos;s from 5&apos; or 3&apos; ends.  Sequences should not begin or end with N.</p>
</li>
<li><p>Remove any low quality sequences.  These have high number of N&apos;s and/or long stretches of polynucleotides throughout. </p>
</li>
</ol>
<h3 id="examples-of-low-quality-sequences">Examples of low quality sequences</h3>
<p>SEQUENCE:</p>
<pre><code>AAAAAAAAAAAAAAAAAAAAACCCCCCCCCCCCCCCCCCCCCCTTTTTTGGGGGG
</code></pre><p>SEQUENCE:</p>
<pre><code>TNNCANNNNGNNNGGNNCNNANNNNNTGCNNNNNNTTNNAANNGCAANNNNTG
</code></pre><h2 id="file-formatting">File Formatting</h2>
<ol>
<li><p>Please send your submission as two text files:</p>
<p>  plc.txt   contains publication, library, and contact files</p>
<p>  est.txt   contains all EST submissions</p>
</li>
<li><p>Ensure that all EST# are unique</p>
</li>
<li><p>EST file CITATION must exactly match Pub file TITLE</p>
</li>
<li><p>EST file LIBRARY must exactly match Lib file NAME</p>
</li>
<li><p>EST file CONT_NAME must exactly match Cont file NAME</p>
</li>
<li><p>PUT_ID field should not contain organism, hypothetical protein, unknown, or similar information</p>
</li>
<li><p>Use only standard 26 letters of the English alphabet (no accent marks, umlauts, etc.)</p>
</li>
<li><p>The following represents the basic structure for each file:</p>
</li>
</ol>
<pre><code>TYPE: Pub

TITLE:

AUTHORS: Lastname,I.I.; Lastname,I.; Lastname,I.I.; etc&#x2026;

YEAR:

STATUS:

||

TYPE: Lib

NAME:

ORGANISM:

DESCR:

||

TYPE: Cont

NAME:

EMAIL:

LAB:

INST:

ADDR:

||

TYPE: EST

STATUS: New

CONT_NAME:

CITATION:

LIBRARY:

PUBLIC:

SEQUENCE:

||
</code></pre><h2 id="data-submission-file-types">Data submission file types</h2>
<p>The batch submission process for <a href="#SubmissionFormatforESTSequenceData">EST sequence data</a> involves the completion of four file types:</p>
<p>a. <a href="#PublicationFiles">Publication</a><br>b. <a href="#LibraryFiles">Library</a><br>c. <a href="#ContactFiles">Contact</a><br>d. <a href="#ESTFiles">EST</a></p>
<p>The format for each file is described below.</p>
<p>If all the ESTs share the same Publication, Library, and Contact
information, you only need to prepare one of each of those files.
Then complete a separate EST file (file type d) for each sequence.</p>
<p>If any of the EST files have different Publication, Library, or Contact information, you must complete a new file of type a, b, or c.</p>
<p>Once we have entered particular Publication, Library, or Contact information into the database, you do not need to resend the data input files.</p>
<h3 id="mailing-files-to-dbest">Mailing files to dbEST</h3>
<p>Send the completed files to: <a href="mailto:batch-sub@ncbi.nlm.nih.gov" target="_blank">batch-sub@ncbi.nlm.nih.gov</a></p>
<p>You can attach all the files to a single email message, or you can include them in the body of the email message. Please be sure that they are in plain text (ASCII) format.</p>
<p>We prefer to have the individual EST data files batched together as much as possible.</p>
<p>You can submit library, publication, and contact data together in one file. You can also send them in the same file as the EST entries - the TYPE field will differentiate them for the parsing software.</p>
<h3 id="assignment-of-genbank-accession-numbers-and-release-of-data">Assignment of GenBank Accession Numbers and release of data</h3>
<p>You will receive a list of dbEST IDs and GenBank accession numbers from a dbEST curator via email.</p>
<p>If you would like your sequences held confidential until publication, you can indicate that by putting the release date in the PUBLIC field of the EST files. Your sequences will be released on that date, or when the accession numbers or sequence data are published, whichever comes first.</p>
<p>Once your sequences are released into the public database, they will be available from GenBank, accessible through <a href="http://www.ncbi.nlm.nih.gov/sites/entrez?db=nucleotide" target="_blank">Entrez Nucleotides</a>.</p>
<h3 id="updating-your-dbest-data">Updating your dbEST data</h3>
<p>Updates to EST entries are done basically in the same way as new entries. Changes to any item in the EST input file (other than EST# or CONT_NAME) are made by completing an input file with new data in the fields that need to be changed. For the STATUS field enter &quot;Update&quot; instead of &quot;New&quot;.</p>
<p>In addition to the fields to be changed Updates need to include TYPE, STATUS, EST#, and CONT_NAME fields.</p>
<p>For changes in Publication, Contact, or Source data, or for changes in EST#&apos;s or CONT_NAME, send an email message describing the change that is needed.</p>
<p>Send the update files to: <a href="mailto:batch-sub@ncbi.nlm.nih.gov" target="_blank">batch-sub@ncbi.nlm.nih.gov</a></p>
<h3 id="questions-and-comments">Questions and Comments</h3>
<p>If you have questions about the EST submission format, please contact <a href="mailto:info@ncbi.nlm.nih.gov" target="_blank">info@ncbi.nlm.nih.gov</a>.  </p>
<h2 id="submission-format-for-est-sequence-data">Submission Format for EST Sequence Data</h2>
<p>The following is a specification for flat file formats for delivering EST and related data to the NCBI EST database.</p>
<ul>
<li>The format consists of colon delineated capitalized field tags, followed by data.</li>
<li>The data fields should appear on the same line as the tag, with no line wrapping. Exceptions to this are the TITLE and AUTHORS fields of the Publication file; Description (DESCR:) field of the Library file; and the CITATION, COMMENT, and SEQUENCE fields of the EST file. In these fields, the data may begin on the same line as the field tag or on the line following the tag and the lines can be wrapped.</li>
<li>Note that some fields are obligatory.</li>
<li>The TYPE field is obligatory at the beginning of each entry, even if there are multiple entries of a given type in a file.</li>
<li>If data is not available for a non-obligatory field, the field can either be omitted entirely, or the tag may be included with an empty data field. Please do not put <code>*</code>, <code>-</code>, etc., to indicate missing data.</li>
<li>Each record (including the last record in the file) should end with a double-bar tag (||) to indicate the end of the record.</li>
<li>Please also see the bulletted tips at the end of each file format for important notes about that file type.</li>
</ul>
<hr>
<h3 id="file-types">File Types</h3>
<p>There are four types of deliverable files:<br>a. <a href="#PublicationFiles">Publication</a><br>b. <a href="#LibraryFiles">Library</a><br>c. <a href="#ContactFiles">Contact</a><br>d. <a href="#ESTFiles">EST</a></p>
<p>Each EST file needs to reference the Publication, Library, and Contact data.  Therefore the Publication, Library, and Contact files must be in the database when the EST file is entered. Once these files have been submitted and entered, they do not need to be re-submitted for additional EST files that have the same Publication, Library, or Contact.</p>
<hr>
<h3 id="publication-files">Publication Files</h3>
<p>These are the valid tags and a short description:</p>
<pre><code>TYPE:      Entry type - must be &quot;Pub&quot; for publication entries. 
           **Obligatory field**
MEDUID:    Medline unique identifier. 
           Not obligatory, include if you know it.
TITLE:     Title of article. (Begin on same line or the line below tag, 
           use multiple lines if necessary)
           **Obligatory field**
AUTHORS:   Author name, format:  Name,I.I.; Name2,I.I.; 
           Name3,I.I.
           (Begin on the same line or the line below tag, use multiple lines if necessary)
           **Obligatory field**
JOURNAL:   Journal name
VOLUME:    Volume number
SUPPL:     Supplement number
ISSUE:     Issue number
I_SUPPL:   Issue supplement number
PAGES:     Page, format:   123-9
YEAR:      Year of publication.
           **Obligatory field**
STATUS:    Publication status.
           1=unpublished, 2=submitted, 3=in press, 4=published
           **Obligatory field**
||
</code></pre><h4 id="examples">Examples</h4>
<pre><code>TYPE: Pub
MEDUID: 92347897
TITLE: 
Expressed sequence tags and chromosomal localization of 
cDNA clones from a subtracted retinal pigment epithelium library
AUTHORS: 
Gieser,L.; Swaroop,A.
JOURNAL: Genomics
VOLUME: 13
ISSUE: 2
PAGES: 873-6
YEAR: 1992
STATUS: 4
||
</code></pre><p>Pub data template with required and most often used fields:</p>
<pre><code>TYPE:      Pub
TITLE:     
title
AUTHORS:   
authors
JOURNAL: 
VOLUME: 
ISSUE:
PAGES: 
YEAR: 
STATUS:    
||
</code></pre><ul>
<li>The TYPE field is obligatory at the beginning of each entry, even if there are multiple entries of a given type in a file.</li>
<li>The MEDUID field is a MEDLINE record unique identifier. We do not normally expect you to supply this - we try to retrieve this from our relational version of MEDLINE database.</li>
<li>For the Publication STATUS field the values are 1=unpublished, 2=submitted, 3=in press, 4=published</li>
<li>The TITLE field is a free format string. The only requirement is that you put an identical string in the CITATION field of the EST files. We match the CITATION in the EST input file automatically against the TITLE for publications in the publication table in order to associate the correct publication with the EST.</li>
</ul>
<hr>
<h3 id="b-library-files">b. <a name="LibraryFiles"></a>Library Files</h3>
<p>These are the valid tags and a short description:</p>
<pre><code>TYPE:       Entry type - must be &quot;Lib&quot; for library entries.
            **Obligatory field**
NAME:       Name of library.
            **Obligatory field**
ORGANISM:   Organism from which library prepared.
            Scientific name.
            **Obligatory field**
STRAIN:     Organism strain
CULTIVAR:   Plant cultivar
ISOLATE:    Individual isolate from which the sequence was obtained
SEX:        Sex of organism (female, male, hermaphrodite)
ORGAN:      Organ name 
TISSUE:     Tissue type
CELL_TYPE:  Cell type
CELL_LINE:  Name of cell line
STAGE:      Developmental stage
HOST:       Laboratory host
VECTOR:     Name of vector.
V_TYPE:     Type of vector (Cosmid, Phage,Plasmid,YAC, other)
RE_1:       Restriction enzyme at site1 of vector
RE_2:       Restriction enzyme at site2 of vector
DESCR:      Description of library preparation methods, vector, 
            etc. 
            Text starts on the same line or the line below the DESCR: tag.
||
</code></pre><h4 id="examples">Examples</h4>
<pre><code>TYPE: Lib
NAME:  Rat embryonic day 17 post-fertilization Library
ORGANISM: Rattus norvegicus
STRAIN: Sprague-Dawley
SEX: male
STAGE: embryonic day 17 post-fertilization
TISSUE: aorta
CELL_TYPE: vascular smooth muscle
DESCR: 
||
</code></pre><p>Lib data template with required and most often used fields:</p>
<pre><code>TYPE:       Lib
NAME:       
ORGANISM:  
STRAIN:   
CULTIVAR: 
SEX:      
ORGAN:   
TISSUE: 
CELL_TYPE: 
CELL_LINE:
STAGE:   
HOST:   
VECTOR:
V_TYPE:  
RE_1:    
RE_2:   
DESCR:      
description
||
</code></pre><ul>
<li>The TYPE field is obligatory at the beginning of each entry, even if there are multiple entries of a given type in a file.</li>
<li>Try to keep the library NAME field to &lt;= 48 characters. We can accept up to 128 characters, but it will be truncated to 48 characters in the identification line of the FASTA file created for BLAST searching.</li>
<li>When you enter the library NAME in the Library Files, please note that the identical string must be used in in the LIBRARY field of the EST Files.</li>
<li>The purpose of the Library file is to describe the source of the sequence. The most important item is the ORGANISM name. A library NAME is required to link the source information to the individual ESTs. The term &apos;Library&apos; does not mean that this information strictly applies only to cDNA libraries.</li>
<li>The DESCR field should contain as much detail about the derivation of the mRNA/cDNA as seems appropriate.</li>
</ul>
<hr>
<h3 id="c-contact-files">c. <a name="ContactFiles"></a>Contact Files</h3>
<p>These are the valid tags and a short description:</p>
<pre><code>TYPE:     Entry type - must be &quot;Cont&quot; for contact entries.
          **Obligatory field**
NAME:     Name of person submitting the EST. 
          **Obligatory field**
FAX:      Fax number as string of digits.
TEL:      Telephone number as string of digits.
EMAIL:    E-mail address
LAB:      Laboratory providing EST.
INST:     Institution name
ADDR:     Address string, comma delineation.
||
</code></pre><h4 id="examples">Examples</h4>
<pre><code>TYPE: Cont
NAME: Sikela JM
FAX: 303 270 7097
TEL: 303 270 
EMAIL: tjs@tally.hsc.colorado.edu
LAB: Department of Pharmacology
INST: University of Colorado Health Sciences Center
ADDR: Box C236, 4200 E. 9th Ave., Denver, CO 80262-0236, USA
||
</code></pre><p>Contact data template with required and most often used fields:</p>
<pre><code>TYPE: Cont
NAME: 
FAX: 
TEL:
EMAIL: 
LAB: 
INST: 
ADDR: 
||
</code></pre><ul>
<li>The TYPE field is obligatory at the beginning of each entry, even if there are multiple entries of a given type in a file.</li>
<li>None of the other fields are obligatory, but we require at least the name of a contact person.</li>
<li>We would like as many of the fields filled in as possible, to provide complete information to the user for contacting a source for the EST or further information about it.</li>
<li>The contact name field (CONT_NAME) in the EST files must contain an identical string to the string used in the NAME field of the Contact file, for automatic matching.</li>
</ul>
<h3 id="d-est-files">d. <a name="ESTFiles"></a>EST Files</h3>
<p>These are the valid tags and a short description:</p>
<pre><code>TYPE:      Entry type - must be &quot;EST&quot; for EST entries.
           **Obligatory field**
STATUS:    Status of EST entry - &quot;New&quot; or &quot;Update&quot;.
           **Obligatory field**
CONT_NAME: Name of contact 
           (must be identical string to the contact entry)
           **Obligatory field**
CITATION:  Journal citation. 
           (Must be identical string to the publication title)
           Begins on the same line or the line below tag - use continuation lines 
           if necessary.
           **Obligatory field**
LIBRARY:   Library name.  
           (Must be identical string to library name entry.)
           **Obligatory field**
EST#:      EST id assigned by contact lab.
           For EST updates, this is the string we match on.
           Length limit: 64 characters.
           **Obligatory field**
GB#:       GenBank accession number
GDB#:      Genome database accession number
GDB_DSEG:  Genome database Dsegment number
CLONE:     Clone id.
SOURCE:    Source providing clone e.g. ATCC
SOURCE_DNA:   Source id number for the clone as pure DNA
SOURCE_INHOST:  Source id number for the clone stored in the host.
OTHER_EST:    Other ESTs on this clone.
DBNAME:    Database name for cross-reference to another database
DBXREF:    Database cross-reference accession
PCR_F:     Forward PCR primer sequence
PCR_B:     Backward PCR primer sequence
INSERT:    Insert length (in bases)
ERROR:     Estimated error in insert length (bases)
PLATE:     Plate number or code
ROW:       Row number or letter
COLUMN:    Column number or letter
SEQ_PRIMER:   Sequencing primer description or sequence.
P_END:     Which end sequenced e.g. 5&apos;
HIQUAL_START:  Base position of start of highest quality sequence 
                (default=1)
HIQUAL_STOP:  Base position of last base of highest quality 
              sequence.
DNA_TYPE:  cDNA (default), Genomic, Viral, Synthetic, Other
PUBLIC:    Date of public release. 
           Leave blank for immediate release.
           Format:  9/11/1994 (MM/DD/YYYY)
            **Obligatory field**
PUT_ID:    Putative identification of sequence by submitter.
TAG_LIB:   Name of library whose tag is found in this sequence.
TAG_TISSUE:  Tissue that was source for the tagged library, if
             a library tag was found.
TAG_SEQ:   The actual sequence of the library tag found in the EST
           read. If the tag was searched for and not found, put
           &apos;Not found&apos; in this field.
POLYA:     Y or N to indicate if a polyA tail was or was not found 
           in the EST sequence.
COMMENT:   Comments about EST. 
           Starts on the same line or the line below COMMENT: tag.
SEQUENCE:  Sequence string. 
           Starts on the same line or the line below SEQUENCE: tag.
           **Obligatory field**
||
</code></pre><h4 id="examples">Examples</h4>
<pre><code>TYPE: EST
STATUS:  New
CONT_NAME: Kerlavage AR
EST#: HHC189f
CLONE: HHC189
SOURCE: ATCC
SOURCE_INHOST: 65128
OTHER_EST:  HHC189r
CITATION: 
Complementary DNA sequencing: expressed sequence tags 
and human genome project
SEQ_PRIMER: M13 Forward
P_END: 5&apos;
HIQUAL_START: 1
HIQUAL_STOP: 285
DNA_TYPE: cDNA
LIBRARY: Hippocampus, Stratagene (cat. #936205)
PUBLIC: 
PUT_ID: Actin, gamma, skeletal
COMMENT:
This is a comment about the sequence.
It may span several lines.
SEQUENCE:
AATCAGCCTGCAAGCAAAAGATAGGAATATTCACCTACAGTGGGCACCTCCTTAAGAAGCTG
ATAGCTTGTTACACAGTAATTAGATTGAAGATAATGGACACGAAACATATTCCGGGATTAAA
CATTCTTGTCAAGAAAGGGGGAGAGAAGTCTGTTGTGCAAGTTTCAAAGAAAAAGGGTACCA
GCAAAAGTGATAATGATTTGAGGATTTCTGTCTCTAATTGGAGGATGATTCTCATGTAAGGT
TGTTAGGAAATGGCAAAGTATTGATGATTGTGTGCTATGTGATTGGTGCTAGATACTTTAAC
TGAGTATACGAGTGAAATACTTGAGACTCGTGTCACTT
||
</code></pre><p>EST data template with required and most often used fields:</p>
<pre><code>TYPE:      EST
STATUS:  
CONT_NAME: 
CITATION:  
publication title
LIBRARY:  
EST#:    
CLONE:  
SOURCE:
SOURCE_DNA:   
SOURCE_INHOST:
PCR_F:     
PCR_B:    
INSERT:  
ERROR:  
PLATE: 
ROW:  
COLUMN:   
SEQ_PRIMER: 
P_END:     
HIQUAL_START: 
HIQUAL_STOP: 
DNA_TYPE: 
PUBLIC: 
PUT_ID:  
POLYA:    
COMMENT:   
comments
SEQUENCE: 
sequence
||
</code></pre><ul>
<li>The <code>TYPE</code> field is obligatory at the beginning of each entry, even if there are multiple entries of a given type in a file.</li>
<li>Valid data values for the <code>EST STATUS</code> field are New (new entry) or Update (change existing <code>EST</code> entry).</li>
<li>When updating an <code>EST</code>, only the fields with text present in the <code>EST</code> file will be changed. The text in the update file will replace entirely the text for that field in the current version of the file.</li>
<li>The <code>DNA_TYPE</code> is assumed to be c<code>DNA</code>, so this field may be omitted unless the <code>DNA</code> type differs from this.</li>
<li>Sequences start on the same line or the line below the <code>SEQUENCE</code> field tag.</li>
<li>It is very important that the strings in the following fields be completely identical, for purposes of automatic matching:</li>
<li><code>POLYA</code> indicates whether a polyA tail sequence was found in the <code>EST</code> read.  </li>
<li><code>TAG_LIB</code>, <code>TAG_TISSUE</code>, <code>TAG_SEQ</code>: these were added because some
   researchers pool libraries to
   make subtracted libraries. It&apos;s important to many people to know
   which of the libraries in the pooled library a particular
   sequence came from, so short sequence tags are being included in
   the library preps. These tags are specific for each library, so
   if you find that tag in a cDNA read, you know it came from that
   library, even though you found it in a library which was a pool
   of libraries.</li>
<li><code>TAG_LIB</code>: contains the name of the library whose tag you found in this sequence. e.g. <code>NCI_CGAP_P</code>. Leave the field blank if you did not find one. If the library name in this field doesn&apos;t match the name of the library the sequence came from, or one of the libraries used to make a subtracted library, this field will alert us that there is a problem.  </li>
<li><code>TAG_TISSUE</code>: contains the tissue that the tagged library was made from e.g. prostate in the above example.  </li>
<li><code>TAG_SEQ</code>: contains the actual tag sequence you found in the <code>EST</code> read. If you didn&apos;t find one, put <code>None found</code> in this field,
to distinguish this case from one where the tag hasn&apos;t been searched for.</li>
</ul>

          

        </div>
      </div>
    

    <div id='shared-content-1' nid='1092'>
      <div class='rightnav'>
        <h2>GenBank Resources</h2>
        <ul>
          <li>
            <a href="/~/">GenBank Home</a></li>
          <li>
            <a href="/~/submit_types">Submission Types</a></li>
          <li>
            <a href="/~/submit">Submission Tools</a></li>
          <li>
            <a href="http://www.ncbi.nlm.nih.gov/nuccore/">Search GenBank</a></li>
          <li>
            <a href="/~/update">Update GenBank Records</a></li>
        </ul>
      </div>
    </div>

  </body>
</html>

