

<html>
  <head>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta charset="UTF-8" />
    <title></title>
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <title>GenBank Submission Tools</title>
  </head>

  <body>

    
      <div class='node clear-block'>
        <div class='content'>

          
            <h1 id="eukaryotic-genome-annotation-guide">Eukaryotic Genome Annotation Guide</h1>
<h2 id="annotation">Annotation</h2>
<p><code>Sequin</code> and <code>tbl2asn</code> use a simple, five-column, tab-delimited table of feature locations and qualifiers to generate annotation.</p>
<p>The format of this feature table allows different kinds of features (e.g. <code>gene</code>, <code>CDS</code> (coding region), <code>tRNA</code>, <code>repeat_region</code>) and qualifiers (e.g. <code>/product</code>, <code>/note</code>) to be indicated. The validator will check for errors such as internal stops in coding regions.</p>
<h2 id="guidelines-for-prokaryotic-genome-submissions-">Guidelines for <a href="genbank/genomesubmit">prokaryotic genome submissions</a> .</h2>
<p>If you do not understand any of the instructions presented here or you have questions, please contact us by email at <a href="mailto:genomes@ncbi.nlm.nih.gov" target="_blank">genomes@ncbi.nlm.nih.gov</a> prior to creating your submission. This will save us both a lot of time.</p>
<h2 id="table-of-contents">Table of Contents</h2>
<ol>
<li><a href="#prepare_table">Prepare annotation table</a><ul>
<li><a href="#Genes">Gene features</a></li>
<li><a href="#%60locus_tag%60"><code>locus_tag</code></a></li>
<li><a href="#%60protein_id%60"><code>protein_id</code></a></li>
<li><a href="#%60transcript_id%60"><code>transcript_id</code></a></li>
<li><a href="#CDS">CDS (coding region) features</a></li>
<li><a href="#Partialcodingregionsinincompletegenomes">Partial coding regions in incomplete genomes</a></li>
<li><a href="#Gene_fragments">Gene fragments</a></li>
<li><a href="#transpliced">Transpliced Genes</a></li>
<li><a href="#Splitgenesontwocontigs">Split genes on two contigs</a></li>
<li><a href="#rRNA">Ribosomal RNA, tRNA and other RNA features</a></li>
<li><a href="#mRNA">mRNA features</a></li>
<li><a href="#Alternativelysplicedgenes">Alternatively spliced genes</a></li>
<li><a href="#Evidence_Qualifiers">Evidence Qualifiers</a></li>
<li><a href="#db_xref">Data base cross references</a></li>
<li><a href="#GOterms">Gene Ontology</a></li>
</ul>
</li>
</ol>
<h2 id="prepare-annotation-table">Prepare annotation table</h2>
<p>The features must be in a simple five-column tab-delimited table, called the feature table. The feature table specifies the location and type of each feature for tbl2asn or Sequin to include in the GenBank submission that is created. The first line of the table contains the following basic information:</p>
<pre><code>&gt;Features SeqID table_name
</code></pre><p>The SeqID must be the same as the sequence&apos;s SeqID in the FASTA file. The table_name is optional. Subsequent lines of the table list the features. Columns are separated by tabs.</p>
<ul>
<li>Column 1: Start location of feature</li>
<li>Column 2: Stop location of feature</li>
<li>Column 3: Feature key</li>
<li>Column 4: Qualifier key</li>
<li>Column 5: Qualifier value</li>
</ul>
<p><a href="~/eukaryotic_genome_submission_Examples#fig2">Figure 2</a> shows a sample feature table and illustrates a number of points about the feature table format. The GenBank flatfile corresponding to this table is shown in <a href="~/eukaryotic_genome_submission_Examples#fig3">Figure 3</a> . The allowed features and their qualifiers are listed in the Feature Table <a href="~/collab/FT/index#7.3.2">documentation</a> .</p>
<p>Features that are on the complementary strand, such as the genes Ngs_3038 and Ngs_11232 and their corresponding features shown in <a href="~/eukaryotic_genome_submission_Examples#fig2">Figure 2</a> , are indicated by reversing the interval locations.</p>
<p>Please avoid unnecessary capitalization in all text entered in your table.</p>
<p>Additional requirements, as well as suggestions for various types of annotation, are included in the following sections.</p>
<h3 id="gene-features">Gene features</h3>
<p>Gene features are always a single interval, and their location should cover the intervals of all the relevant features such as promoters and polyA binding sites.</p>
<p>Gene names should follow the standard nomenclature rules of the particular organism. For example, mouse gene names begin with an uppercase letter, and the remaining letters are lowercase.</p>
<p>Coding regions (CDS) and RNAs, such as tRNAs and rRNAs, must have a corresponding gene feature. However, other features such as repeat_regions and misc_features do not have a corresponding gene or <code>locus_tag</code>.</p>
<h3 id="locustag"><code>locus_tag</code></h3>
<p>All genes should be assigned a systematic gene identifier which should receive the <code>locus_tag</code> qualifier on the gene feature in the table. Genes may also have functional names as assigned in the scientific literature. In this example, KCS_0001 is the systematic gene identifier, while Abc5 is the functional gene name.</p>
<h3 id="table-view-of-gene-with-both-biological-name-and-locustag">Table view of gene with both biological name and <code>locus_tag</code></h3>
<pre><code>1       1575    gene
                        gene    Abc5
                        `locus_tag`        KCS_0001
</code></pre><h3 id="flatfile-view">Flatfile view</h3>
<pre><code>gene       1..1575
                     /gene=&quot;Abc5&quot;
                     /`locus_tag`=&quot;KCS_0001&quot;
</code></pre><h3 id="table-view-of-gene-with-only-locustag">Table view of gene with only <code>locus_tag</code></h3>
<pre><code>1 1575    gene
                        `locus_tag`     KCS_0001
</code></pre><h3 id="flatfile-view">Flatfile view</h3>
<pre><code>gene       1..1575
                     /`locus_tag`=&quot;KCS_0001&quot;
</code></pre><p>For consistency the same <code>locus_tag</code> prefix must be used throughout the entire genome. Therefore, all the chromosomes of a genome should have the same <code>locus_tag</code> prefix.</p>
<p>To improve the use of <code>locus_tag</code>s we are now requiring that all <code>locus_tag</code> prefixes be registered and that they be unique. We recommend having the BioProject registration process auto-assign a <code>locus_tag</code> prefix, as they are not meant to convey meaning. The <code>locus_tag</code> prefix should be 3-12 alphanumeric characters and the first character may not be a digit. The <code>locus_tag</code> prefix is followed by an underscore and then an alphanumeric identification number that is unique within the given genome. Other than the single underscore used to separate the prefix from the identification number, no special characters can be used in the <code>locus_tag</code>.</p>
<p>The chromosome number can be embedded in the <code>locus_tag</code>, if desired, in the format <code>Prefix\_</code><em>#g#####</em>, where the first <code>#</code> is the chromosome number and <code>#####</code> is the unique number of the gene. For example, <code>Ajs_4g00123</code> for a gene on chromosome 4.</p>
<p>Read more about <a href="http://www.ncbi.nlm.nih.gov/genomes/locustag/Proposal.pdf" target="_blank"><code>locus_tag</code>s</a> and their intended usage.</p>
<p>Please register your genome project and proposed <code>locus_tag</code> prefix on the <a href="https://submit.ncbi.nlm.nih.gov/subs/bioproject/" target="_blank">BioProject registration</a> page prior to preparing your submission to GenBank. Each project that is registered here is assigned a project_id, and in the future we intend that the project_id will appear on all entries associated with a particular genome project.</p>
<p>The use of <code>locus_tag</code> is supported in Sequin version 4.35 or later. If you have an older version of Sequin please download the <a href="http://www.ncbi.nlm.nih.gov/Sequin/" target="_blank">current version</a> . The latest version of tbl2asn is available from the <a href="~/tbl2asn2">tbl2asn</a> page.</p>
<h3 id="proteinid"><code>protein_id</code></h3>
<p>All proteins in a WGS or complete genome must be assigned an identification number by the submitter. We use this number to track proteins when sequences are updated. This number is indicated in the table by the CDS qualifier <code>protein_id</code>, and should have the format gnl|dbname|string, where dbname is a version of your lab name that you think will be unique (eg SmithUCSD), and string is the unique protein SeqID assigned by the submitter. This identifier is saved with the record (in ASN.1 format), but it is not visible in the flatfile. We recommend using the <code>locus_tag</code> as the protein SeqID. In this example, the <code>protein_id</code> for <code>ABC5</code> is <code>gnl|SmithUCSD|KCS_0001</code>.</p>
<h4 id="example">Example:</h4>
<pre><code>&lt;1      &gt;1575        gene
                        gene    Abc5
                        locus_tag    KCS_0001
1       1575    CDS
                        product      ABC5
                        protein_id   gnl|SmithUCSD|KCS_0001
</code></pre><p>Since the <code>protein_id</code> is used for internal tracking in our database, it is important that the complete <code>protein_id</code> (dbname + SeqID) not be duplicated by a genome center. Thus, if your genome center is submitting more than one complete genome, please be sure to use unique <code>protein_id</code>&apos;s for all of the genomes.</p>
<p>The <code>protein_id</code> is also included as a qualifier on the corresponding mRNA feature, to allow the CDS and mRNA to be paired during processing.</p>
<p>Note that when WGS submissions are processed, the dbname in the <code>protein_id</code> is automatically changed to &apos;WGS:XXXX&apos;, where XXXX is the project&apos;s accession number prefix.</p>
<p>After your genome is released into GenBank, the proteins are assigned accession numbers. We will provide a table of the protein SeqIDs and accession numbers for you to use in future <a href="~/eukaryotic_genome_submission#updating">updates</a> .</p>
<h3 id="transcriptid"><code>transcript_id</code></h3>
<p>The <code>transcript_id</code> is included as a qualifier for both the CDS and its corresponding mRNA. It has the same format as the <code>protein_id</code>, gnl|dbname|identifier. Because each <code>transcript_id</code> and <code>protein_id</code> must be unique, we suggest adding &apos;mrna&apos; or &apos;t&apos; to the <code>protein_id</code> identifier as a simple way to create the corresponding (unique) <code>transcript_id</code>. However, you can use whatever naming convention you choose, as long as all of the identifiers are unique.</p>
<pre><code>63574        87173   gene
                        `locus_tag`     Ngs_17131
63574   63907   mRNA
75690   75730
84396   85536
85598   85773
85836   86109
86173   86467
86555   86670
86731   87173
                        product     hypothetical protein
                        protein_id  gnl|ncbi|Ngs_17131
                        transcript_id     gnl|ncbi|mrna.Ngs_17131
84402   85536   CDS
85598   85773
85836   86109
86173   86467
86555   86670
86731   86882
                        product     hypothetical protein
                        protein_id  gnl|ncbi|Ngs_17131
                        transcript_id     gnl|ncbi|mrna.Ngs_17131
</code></pre><h3 id="cds-coding-region-features"><code>CDS</code> (coding region) features</h3>
<p>All CDS features must have a product qualifier (protein name). NCBI protein naming conventions are adopted in part from the <a href="http://www.uniprot.org/docs/gennameprot" target="_blank">UniProt-Swiss-Prot Protein Knowledgebase</a></p>
<p>Consistent nomenclature is indispensable for communication, literature searching and data retrieval. Many species-specific communities have established gene nomenclature committees that try to assign consistent and, if possible, meaningful gene symbols. Other scientific communities have established protein nomenclatures for a set of proteins based on sequence similarity and/or function. But there is no established organization involved in the standardization of protein names, nor are there any efforts to establish naming rules that are valid across the largest spectrum of species possible.</p>
<p>Ambiguities regarding gene/protein names are a major problem in the literature and it is even worse in the sequence databases which tend to propagate the confusion. For this reason, we ask that you follow some basic guidelines in naming your proteins. The protein naming guidelines are based on the premise that a good and stable recommended name for a protein is a name that is as neutral as possible.</p>
<h4 id="guidelines-for-naming-proteins">Guidelines for naming proteins:</h4>
<ul>
<li>If it exists, use the approved nomenclature.</li>
<li>Use a concise name, not a description or phrase.</li>
<li>Ideally the name should be unique and attributed to all orthologs.</li>
<li>The protein name should not contain specific characteristics of the protein, and it should not reflect the function of the protein, its subcellular location, its domain structure, its molecular weight or its species of origin. This information can be included in the note.</li>
<li>In cases where the protein name is not known use &quot;unknown&quot; or &quot;hypothetical protein&quot; as the product name. We recommend the use of &quot;hypothetical protein&quot; as this will allow the <code>locus_tag</code> identifier to be appended to the product name in BLAST and Entrez summary lines.</li>
<li>Protein names may be denoted by the same symbol as the corresponding gene, but in the correct format for the organism. For example, mouse proteins have the same symbol as the gene name, but the protein name has all capital letters.</li>
<li>Avoid the use of molecular weights in protein names; &quot;unicornase subunit A&quot; is preferred to &quot;unicornase 52 kDa subunit&quot;</li>
<li>Do not use the term &quot;homolog&quot; in a protein as this infers an evolutionary relationship that has generally not been determined.</li>
<li>Where possible, avoid the use of commas in protein names</li>
<li>Use lowercase letters, except when uppercase are required (for example, in acronyms such as DNA or ATP).</li>
<li>Wherever appropriate, the name should use American spelling conventions.</li>
<li>Avoid the use of Roman numerals where possible. Use instead Arabic numbers.</li>
<li>Do not build molecular weights into abbreviations</li>
<li>For proteins that belong to a multigene family, it is recommended that you choose a coherent nomenclature with numbers to specify the different members of the family.</li>
<li>When naming proteins which can be grouped into a family based on homology or according to a notion of shared function, the different members should be enumerated with a dash &quot;-&quot; followed by an Arabic number. e.g. &quot;desmoglein-1&quot;, &quot;desmoglein-2&quot;, etc.</li>
<li>Greek letters must be written in full e.g. &quot;alpha&quot;, and written entirely in lower case with the exception of &quot;Delta&quot; in the context of steroid/fatty acid metabolism nomenclature. Additionally the Greek letters that are followed by a number should be preceded or followed by a dash &quot;-&quot; e.g. &quot;unicornase alpha-1&quot;.</li>
<li>Do not use diacritics, such as accents, umlauts. Many computer sytems (ours included) can only understand ASCII characters.</li>
<li>Do not use plurals in a protein name. e.g. &quot;ankyrin repeats-containing protein 8&quot; is wrong.</li>
<li>Proteins of unknown function which contain a defined domain or motif, can be named according to the domain present. The name should be of the following type: &quot;<domain|repeat>-containing protein&quot;. e.g. &quot;PAS domain-containing protein 5&quot;.</domain|repeat></li>
</ul>
<p>Here are some examples of good protein names:</p>
<dl class="goodbad">

<dt>**cytochrome b**</dt>

<dt>**CytB**</dt>

<dt>**aconitate hydrase B**</dt>

<dt>**hypothetical protein**</dt>

<dt>**cytochrome b-like protein**</dt>

<dt>**4Fe-4S cluster binding protein**</dt>

<dt>**adenylyltransferase/ADP-heptose synthase**</dt>

<dt>**2-hydroxyhepta-2,4-diene-1,7-dioate isomerase**</dt>

<dt>**short-chain specific acyl-CoA dehydrogenase**</dt>

<dt>**formylmethanofuran--tetrahydromethanopterin formyltransferase**</dt>

<dt>**serine/threonine-protein kinase**</dt>

<dt>**translation initiation factor 1**</dt>

<dt>**triphosphoribosyl-dephospho-CoA synthetase**</dt>

<dt>**thiamine biosynthesis protein ThiC**</dt>

<dt>**PAS domain-containing protein 5**</dt>

<dt>**ABC transporter ATP-binding protein AlbC**</dt>

<dt>**stage 0 sporulation protein J**</dt>

<dd>_These names all concisely describe the function of the protein, where known, and avoid references to structure, homology and species._</dd>

</dl>

<p>Here are some examples of bad protein names:</p>
<dl class="goodbad">

<dt>**required for the efficient incorporation of molybdate into molybdoproteins**</dt>

<dd>_This describes the protein&apos;s role in a biosynthetic process but is not a protein name._</dd>

<dt>**chaperone Hsp70; DNA biosynthesis; autoregulated heat shock proteins**</dt>

<dd>_The name &quot;chaperone Hsp70&quot; is fine however the remaining comments would be best fielded as a note or in the function qualifier._</dd>

<dt>**putative carbonic anhdrase (EC 4.2.1.1)**</dt>

<dd>_The EC number should not be part of the protein name but instead fielded in the EC_number qualifier_</dd>

<dt>**similar to aconitrate hydrase B**</dt>

<dd>_This statement is fine as a note, however as a protein name aconitrate hydrase B-like protein is preferred_</dd>

<dt>**related to protein of unknown function**</dt>

<dd>_uninformative name_</dd>

<dt>**cytochrome b-like**</dt>

<dd>_cytochrome b-like protein is preferred_</dd>

<dt>**ABC transporter related**</dt>

<dd>_vague name, there are many ABC transporters and subunits, be more specific_</dd>

<dt>**pirin, N-terminal:pirin, C-terminal**</dt>

<dd>_uniformative name noting similarity in N and C terminus_</dd>

<dt>**helix-turn-helix motif**</dt>

<dd>_Describes a motif or structural domain but is not an appropriate protein name._</dd>

<dt>**PP-loop**</dt>

<dd>_Describes a motif or structural domain but is not an appropriate protein name._</dd>

<dt>**alpha/beta hydrolast fold**</dt>

<dd>_Describes a motif or structural domain but is not an appropriate protein name._</dd>

<dt>**pentapeptide repeat**</dt>

<dd>_Describes a motif or structural domain but is not an appropriate protein name._</dd>

<dt>**phosphopantetheine-binding domain**</dt>

<dd>_Describes a motif or structural domain but is not an appropriate protein name._</dd>

<dt>**protein of unknown function:conserved**</dt>

<dd>_uninformative name_</dd>

<dt>**hypothetical 32.5 kDa protein homologous to phytoene and squalene synthethases**</dt>

<dd>_Hypothetical protein alone is appropriate. The remaining comments should be fielded as a note._</dd>

<dt>**ribosomal protein L3 (E. coli)**</dt>

<dd>_Protein names should not contain references to organism names. Ribosomal protein L3 is an appropriate name by itself._</dd>

<dt>**saccharopine dehydrogenase or related protein**</dt>

<dd>_&quot;saccharopine dehydrogenase&quot; or &quot;saccharopine dehydrogenase-like protein&quot; would be more appropriate_</dd>

<dt>**tyrosine-protein kinase (capsular polysaccharide biosynthesis)**</dt>

<dd>_tyrosine-protein kinase is fine as a protein name but capsular polysaccharide biosynthesis would be more appropriate as a function._</dd>

<dt>**RimM protein, required for 16S rRNA processing**</dt>

<dd>_RimM is fine as a protein name but descriptive comments should be placed in the note._</dd>

<dt>**involved in flagellar biosynthesis**</dt>

<dd>_This is a functional comment and not a protein name._</dd>

</dl>

<h4 id="notes">Notes</h4>
<p>Please avoid including notes indicating a specific percentage of similarity to other entries in the database, since the corresponding record that you have pointed to may change and make your current note inaccurate, incorrect and obsolete. Descriptions, notes describing similarity to other proteins, and functional comments must be placed in the appropriate CDS qualifiers such as note, or prot_desc, as they are descriptors of the product. E.C. numbers must be fielded in an EC_number qualifier.</p>
<pre><code>start      stop    CDS
                        product     DNA gyrase subunit B
                        EC_number   5.99.1.3
</code></pre><p>Qualifiers that can be used on the CDS feature are:</p>
<pre><code>start    stop    CDS
                        product
                        prot_desc
                        function
                        EC_number
                        note
                        experiment
                        inference
                        go_component
                        go_process
                        go_function
                        db_xref
                        pseudo
                        exception
                        transl_except
</code></pre><p>Multiple <code>note</code> qualifiers are supported. tbl2asn or Sequin concatenate multiple <code>note</code> qualifiers into a single <code>note</code> with semicolons as separators.</p>
<h4 id="bifunctional-proteins">Bifunctional proteins:</h4>
<p>If a protein contains two separate and distinct functions or if it has more than one name, these can be annotated in several ways, as outlined below. Table view:</p>
<pre><code>start  stop    CDS
                        product     adenylyltransferase/ADP-heptose synthase
                        note   bifunctional
</code></pre><p>or</p>
<pre><code>start    stop    CDS
                        product     bifunctional adenylyltransferase/ADP-heptose synthase cyclohydrolase
</code></pre><p>or</p>
<pre><code>start    stop    CDS
                        product     FolD 
                        function    adenylyltransferase
                        function    ADP-heptose synthase cyclohydrolase
                        note   bifunctional
</code></pre><h3 id="partial-coding-regions-in-incomplete-genomes">Partial coding regions in incomplete genomes</h3>
<p>To annotate an partial coding region, you should use the &quot;&lt;&quot; or &quot;&gt;&quot; in your feature table to designate the feature as either 5&apos; or 3&apos; partial. The coding region should begin at the first nucleotide present, however the translation will start at the first complete codon.</p>
<h4 id="example">Example</h4>
<p>In the first example below, the &quot;&lt;&quot; designates this coding region as 5&apos; partial and &quot;codon_start 3&quot; tells the software to start translation with the third nucleotide of the CDS. Note that if the codon_start is not specified, then the software assumes a codon_start of 1. The second coding region below is partial at the 3&apos; end so &quot;&gt;&quot; is used to indicate a 3&apos; partial feature.</p>
<pre><code>&lt;1  497     CDS
                        product       ABC5
                        note   similar to Bacillus subtilis aldolase
                        codon_start  3
                        protein_id   gnl|dbname|KCS_0001
                        transcript_id   gnl|dbname|KCS_mrna0001

600     &gt;1575        CDS
                        product         actin-like protein
                        protein_id      gnl|dbname|KCS_0002
                        transcript_id   gnl|dbname|KCS_mrna0002
</code></pre><p>Here are <a href="~/examples.wgs#partialcds">more examples of formatting partial CDS features</a> .</p>
<p>Partial genes within a sequence should begin or end at consensus splice sites.</p>
<p>The use of partial coding regions is reserved for incomplete genomes only. All coding regions annotated on finished genomes should be complete at both the 5&apos; and 3&apos; ends.</p>
<h4 id="gene-fragments">Gene fragments</h4>
<p>Sometimes a genome will have adjacent or nearby genes that seem to be only part of a protein. In many cases these indicate a possible problem with the sequence and/or annotation. A related issue is the presence of internal stop codons in the conceptual translation of a CDS that looks like it should be a real CDS. These problems may be due to a variety of reasons, including mutations or sequencing artifacts. They can be annotated in a number of ways:</p>
<ol>
<li>Annotate the gene as a pseudogene. If multiple gene fragments were present initially, then add a single gene feature which covers all of the potential coding regions and add the pseudo qualifier indicating that this is a pseudogene. If known, a note qualifier may be added indicating why this gene is disrupted.</li>
</ol>
<pre><code>1    200     gene
                            gene    Abc5
                            `locus_tag`        KCS_0001
                            gene_desc       alkaline phosphatase
                            pseudo
                            note    frameshift
</code></pre><ol>
<li>Alternatively, if you are not sure if the disrupted gene is a &quot;pseudogene&quot; you can just use a gene feature without the /pseudo. Please use the complete nucleotide spans of the frameshifted gene. If known, the reason for the incomplete translation can be included in a note.</li>
</ol>
<pre><code>1  200     gene
                            gene    Abc5
                            `locus_tag`        KCS_0001
                            gene_desc       alkaline phosphatase
                            note    nonfunctional due to frameshift
</code></pre><ol>
<li>If the feature is just noting a similarity to genes in the database and is probably not translated, then it should be annotated as a misc_feature without a corresponding gene feature.</li>
</ol>
<pre><code>1  200     misc_feature
                            note    similar to Abc5
</code></pre><h4 id="transpliced-genes">Transpliced Genes</h4>
<p>Transpliced genes are the exception to the rule for annotating gene feature spans. Transpliced genes are similar to intron-containing genes, except that the two pieces of the gene occur on different regions of the chromosome. These genes are transcribed as two or more separate RNA products that are transpliced into a single mRNA or tRNA. To annotate this using a table, enter the nucleotide spans so that the complementary (minus strand) spans are arranged from high to low, and vice-versa for the plus strand.</p>
<pre><code>36700   36618   gene
86988   87064
                        `locus_tag`    NEQ_t38
                        exception    trans-splicing
36631   36618   misc_feature
                        note    sequence cleaved during processing of trans-spliced tRNAs
36673   36635
87030   87064   tRNA
                        product tRNA-Glu
                        exception    trans-splicing
                        note    this trans-spliced tRNA consists of two halves on mixed strands; it shares a 3&apos; half with another tRNA
</code></pre><pre><code>gene            join(complement(36618..36700),86988..87064)
                     /`locus_tag`=&quot;NEQ_t38&quot;
                     /trans_splicing
     misc_feature    complement(36618..36631)
                     /`locus_tag`=&quot;NEQ_t38&quot;
                     /note=&quot;sequence cleaved during processing of trans-spliced tRNAs&quot;
     tRNA            join(complement(36635..36673),87030..87064)
                     /`locus_tag`=&quot;NEQ_t38&quot;
                     /product=&quot;tRNA-Glu&quot;
                     /trans_splicing
                     /note=&quot;this trans-spliced tRNA consists of two halves on
                     mixed strands; it shares a 3&apos; half with another tRNA&quot;
</code></pre><h4 id="split-genes-on-two-contigs">Split genes on two contigs</h4>
<p>Sometimes, in incomplete genomes, the ends of a gene may be on different contigs. When you are certain that the two pieces are part of the same gene, annotate these as separate genes with unique <code>locus_tag</code> features, plus separate <code>CDS</code>/<code>mRNA</code> features with different <code>protein_id</code> and <code>transcript_id</code> qualifiers. In addition, link the features together with <code>note</code> qualifiers that refer to the other part of the gene. However, do not create extremely short features; for example, if one end is only the start methionine, or only a few amino acids before the stop codon.</p>
<h4 id="example">Example</h4>
<pre><code>&gt;Feature Cont01.00111
5000    &gt;7500        gene
                        `locus_tag`     KCS_2223A
5000    5500    mRNA
6000    &gt;7200
                        product       enolase
                        protein_id    gnl|dbname|KCS_2223A
                        transcript_id    gnl|dbname|KCS_mrna2223A
5488    5500    CDS
6000    &gt;7200
                        product       enolase
                        protein_id    gnl|dbname|KCS_2223A
                        transcript_id    gnl|dbname|KCS_mrna2223A
                        note   5&apos; end; 3&apos; end is gene KCS_2223B on contig Cont01.00224
</code></pre><pre><code>&gt;Feature Cont01.00224
&lt;1   1000    gene    
                        `locus_tag`    KCS_2223B
&lt;100 1000    mRNA
                        product      enolase
                        protein_id   gnl|dbname|KCS_2223B
                        transcript_id    gnl|dbname|KCS_mrna2223B
&lt;100 876     CDS
                        product      enolase
                        protein_id   gnl|dbname|KCS_2223B
                        transcript_id    gnl|dbname|KCS_mrna2223B
                        note   3&apos; end; 5&apos; end is gene KCS_2223A on contig Cont01.00111
</code></pre><h4 id="ribosomal-rna-trna-and-other-rna-features">Ribosomal RNA, tRNA and other RNA features</h4>
<p>RNA features (rRNA, tRNA, ncRNA) need a corresponding gene feature with a <code>locus_tag</code> qualifier. If the amino acid of a tRNA is unknown, use tRNA-Xxx as the product, as in the example. Many submitters like to label the tRNAs such as tRNA-Gly1, etc. If you wish to do this please include &quot;tRNA-Gly1&quot; as a note and not in /gene. The use of /gene is reserved for the actual biological gene symbol such as &quot;trnG&quot;. If a tRNA is a pseudogene, please use the /pseudo qualifier.</p>
<p>Annotate ncRNAs that belong to one of the INSDC <a href="http://www.insdc.org/documents/ncrna-vocabulary" target="_blank">nRNA_class</a> as an ncRNA feature, with the appropriate value in the required /ncRNA_class qualifier. Regions of an RNA should be annotated as a misc_feature (eg, leader sequences), or a misc_binding feature if they bind a known molecule (eg, riboswitches). If the RFAM identifier is known, it can be included as a <a href="#db_xref">db_xref</a>.</p>
<p>Some rRNA, tRNA and ncRNA examples:</p>
<pre><code>&lt;1      400     gene
                        `locus_tag`       KCS_00011
&lt;1      400     rRNA
                        product 16S ribosomal RNA
488     560     gene
                        `locus_tag`       KCS_00012
488     560     tRNA
                        product tRNA-Lys
570     601     gene
                        `locus_tag`       KCS_00020
                        pseudo
570     601     tRNA
                        product tRNA-Phe
                        pseudo
700     780     gene
                        `locus_tag`       KCS_00013
700     780     tRNA
                        product tRNA-Xxx
900     923     gene
                        `locus_tag`       KCS_00014
900     923     ncRNA
                        ncRNA_class     miRNA
                        product mir-9c
950     1000    gene
                        `locus_tag`       KCS_00015
950     1000    tmRNA
                        product tmRNA
</code></pre><p><code>misc_feature</code> and <code>misc_binding</code> features do not have an associated <code>gene</code> feature. To tag these features with a <code>locus_tag</code>-like identifier, include that value in the <code>note</code>, separated from other information by a semicolon and space.</p>
<p>Riboswitches should be annotated using the <code>misc_binding</code> feature if the bound moiety is known; for example:</p>
<pre><code>1     100    misc_binding
                        note cobalamin riboswitch
                        bound_moiety adenosylcobalamin
</code></pre><p>If the bound moiety is unknown or if the sequence is a leader sequence, annotate as a `misc_feature, for example:</p>
<pre><code>1     100    misc_feature
                        note yybP-ykoY element
</code></pre><h4 id="mrna-features">mRNA features</h4>
<p>Include an mRNA feature for each translated CDS. Several things to note are:</p>
<ul>
<li>Use the same product name for the mRNA and its corresponding CDS.</li>
<li>If there is no UTR information, then the mRNA&apos;s location will agree with its CDS&apos;s location, but the mRNA will be partial at its 5&apos; and 3&apos; ends.</li>
<li>Extend the gene feature to include the entire mRNA.</li>
<li>If the mRNA is partial, then make the gene partial.</li>
</ul>
<h4 id="example">Example</h4>
<p>The first example is a complete <code>CDS</code> whose 5&apos; and 3&apos; UTRs are known.</p>
<pre><code>&gt;Feature Cont54
10400   12512   gene
                        locus_tag    CCC_03116
10400   10462   mRNA
10533   10577   
10651   11098   
11182   11642   
11716   12512   
                        product     hypothetical protein
                        protein_id  gnl|dbname|CCC_03116
                        transcript_id    gnl|dbname|CCC_mrna03116
10450   10462   CDS
10533   10577   
10651   11098   
11182   11642   
11716   12233   
                        product     hypothetical protein
                        protein_id  gnl|dbname|CCC_03116
                        transcript_id    gnl|dbname|CCC_mrna03116
</code></pre><p>The second example is a <code>CDS</code> that is partial at the 5&apos; end and lacks any 3&apos; UTR information.</p>
<pre><code>&gt;Feature Cont3
&lt;1   &gt;497 gene             
                        locus_tag CCC_111011
&lt;1   497     CDS             
                        note    similar to Bacillus subtilis aldolase
                        product   aldolase-like protein
                        codon_start     3
                        protein_id   gnl|dbname|CCC_111011
                        transcript_id    gnl|dbname|CCC_mrna111011
&lt;1   &gt;497 mRNA             
                        product   aldolase-like protein
                        protein_id   gnl|dbname|CCC_111011
                        transcript_id    gnl|dbname|CCC_mrna111011
</code></pre><h3 id="alternatively-spliced-genes">Alternatively spliced genes</h3>
<p>In many cases a gene can be alternatively spliced, yielding alternative transcripts. These transcripts may differ in the coding region and produce different products, or they may differ in the non-translated 5&apos; or 3&apos; UTR and produce the same protein. To annotate alternatively spliced genes, include one <code>mRNA</code> and <code>CDS</code> for each transcript, and include only one gene over all of the features. Give the corresponding <code>mRNA</code> and <code>CDS</code> the same name, and include a <code>note</code> with the value <code>alternatively spliced</code> on each. If there are multiple <code>CDS</code> with the same name, then add a <code>note</code> to each <code>mRNA</code> and <code>CDS</code> to refer to each other; e.g., <code>transcript variant A</code> and <code>encoded by transcript variant A</code> for one <code>mRNA</code>/<code>CDS</code> pair. If the <code>CDS</code> have different translations, then they should have different product names. Make sure that all the proteins have unique <code>protein_id</code> values.</p>
<h4 id="examples">Examples</h4>
<p>Example 1 (different products):</p>
<pre><code>&gt;Feature Cont01.00055
10      5000    gene    
                        locus_tag    CCC_04562
10      500     mRNA
722     1555
2548    3901
4400    5000
                        product     CCC_04562 isoform A
                        note alternatively spliced
                        protein_id  gnl|dbname|CCC_04562A
                        transcript_id   gnl|dbname|CCC_mrna04562A
102     500     CDS
722     1555
2548    3901
4400    4566
                        product    CCC_04562 isoform A
                        note alternatively spliced
                        protein_id  gnl|dbname|CCC_04562A
                        transcript_id   gnl|dbname|CCC_mrna04562A
10      500     mRNA
2548    3901
4400    5000
                        product     CCC_04562 isoform B
                        note alternatively spliced
                        protein_id  gnl|dbname|CCC_04562B
                        transcript_id   gnl|dbname|CCC_mrna04562B
102     500     CDS
2548    3901
4400    4566
                        product     CCC_04562 isoform B
                        note alternatively spliced
                        protein_id  gnl|dbname|CCC_04562B
                        transcript_id   gnl|dbname|CCC_mrna04562B
</code></pre><p>Example 2 (same product):</p>
<pre><code>&gt;Feature Cont01.00056
100     1000    gene
                        locus_tag    CCC_03222
100     333     mRNA
444     678
800     1000
                        product     hypothetical protein
                        note    transcript variant A; alternatively spliced
                        protein_id  gnl|dbname|CCC_03222A
                        transcript_id   gnl|dbname|CCC_mrna03222A
456     678     CDS
800     865
                        product     hypothetical protein
                        note    encoded by transcript variant A; alternatively spliced
                        protein_id  gnl|dbname|CCC_03222A
                        transcript_id   gnl|dbname|CCC_mrna03222A
100     360     mRNA
444     678
800     1000
                        product     hypothetical protein
                        note    transcript variant B; alternatively spliced
                        protein_id  gnl|dbname|CCC_03222B
                        transcript_id   gnl|dbname|CCC_mrna03222B
456     678     CDS
800     865
                        product     hypothetical protein
                        note    encoded by transcript variant B; alternatively spliced
                        protein_id  gnl|dbname|CCC_03222B       
                        transcript_id   gnl|dbname|CCC_mrna03222B
</code></pre><h3 id="evidence-qualifiers">Evidence Qualifiers</h3>
<p>The International Nucleotide Sequence Database Collaboration, DDBJ, EMBL and GenBank has adopted a set of new qualifiers to describe the evidence for feature annotation in GenBank records. These are:</p>
<p>/experimental=&quot;text&quot; /inference=&quot;TYPE:text&quot;, where &apos;TYPE&apos; is from a select list and &apos;text&apos; is structured text.</p>
<p>The use of the new evidence qualifiers is supported in Sequin version 6.0 or later, and tbl2asn version 5.1 or later. If you have an older version, please download the current version of <a href="http://www.ncbi.nlm.nih.gov/Sequin/" target="_blank">Sequin</a> or <a href="~/tbl2asn2">tbl2asn</a> .</p>
<p>These new qualifiers replace /evidence=experimental and /evidence=non-experimental, respectively, which will no longer be supported.</p>
<p>See more information about the <a href="~/evidence">Evidence Qualifiers</a> .</p>
<h4 id="database-cross-references">Database cross references</h4>
<p>A variety of database cross references can be added to a feature. These appear as /db_xref on the features. This qualifier serves as a vehicle for linking of sequence records to other external databases. See the full list of <a href="~/collab/db_xref">db_xref</a> databases.</p>
<pre><code>1     100   CDS
                product     RecA
                protein_id  gnl|center_name|Test_0001
                db_xref InterPro:IPR000111

180     210    misc_feature
                        note yybP-ykoY element
                        db_xref RFAM:RF00080
</code></pre><h4 id="gene-ontology">Gene Ontology</h4>
<p>GO (Gene Ontology) terms can be included in genomes in order to describe protein functionality. Gene Ontology (GO) terms can be indicated with the following qualifiers</p>
<pre><code>1       100     CDS
                        product helicase
                        go_process      chromatin assembly or disassembly|0006333||IEA
                        go_process      antimicrobial humoral response|0019730|16163390|IMP
                        go_component    nucleus|0005634|14668392|IDA
                        go_component    chromatin|0000785||IEA
                        go_function     ATP-dependent helicase activity|0008026||ISS
                        go_function     nucleic acid binding|0003676||IEA
                        go_function     ATP binding|0005524||IEA
</code></pre><p>The value field is separated by vertical bars &apos;|&apos; into a descriptive string, the GO identifier (leading zeroes are retained), and optionally a PubMed ID and one or more evidence codes. The evidence code is the fourth token, so include blank fields, as necessary (eg the last qualifier has no PubMed ID so the third field is blank).</p>

          

        </div>
      </div>
    

    <div id='shared-content-1' nid='1092'>
      <div class='rightnav'>
        <h2>GenBank Resources</h2>
        <ul>
          <li>
            <a href="/~/">GenBank Home</a></li>
          <li>
            <a href="/~/submit_types">Submission Types</a></li>
          <li>
            <a href="/~/submit">Submission Tools</a></li>
          <li>
            <a href="http://www.ncbi.nlm.nih.gov/nuccore/">Search GenBank</a></li>
          <li>
            <a href="/~/update">Update GenBank Records</a></li>
        </ul>
      </div>
    </div>

  </body>
</html>

