

<html>
  <head>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta charset="UTF-8" />
    <title></title>
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <title>GenBank Submission Tools</title>
  </head>

  <body>

    
      <div class='node clear-block'>
        <div class='content'>

          
            <h1 id="prokaryotic-genome-annotation-guide">Prokaryotic Genome Annotation Guide</h1>
<h3 id="annotation">Annotation</h3>
<p>Sequin and tbl2asn use a simple five-column tab-delimited table of feature locations and qualifiers in order to generate annotation.</p>
<p>The format of this feature table allows diferent kinds of features (e.g. gene, coding region, tRNA, repeat_region) and qualifiers (e.g. /product, /note) to be indicated. The validator will check for errors such as internal stops in coding regions.</p>
<p>Guidelines for <a href="~/eukaryotic_genome_submission">eukaryotic genome submissions</a> .</p>
<p>If you do not understand any of the instructions presented here or you have questions, please contact us by email at <a href="mailto:genomes@ncbi.nlm.nih.gov" target="_blank">genomes@ncbi.nlm.nih.gov</a> prior to creating your submission. This will save us both a lot of time.</p>
<h3 id="table-of-contents">Table of Contents</h3>
<ol>
<li><a href="#prepare_table">Prepare annotation table</a><ul>
<li><a href="#Gene_features">Gene features</a></li>
<li><a href="#locus_tag">locus_tag is required</a></li>
<li><a href="#protein_id">protein_id</a></li>
<li><a href="#CDS">CDS (coding region) features and protein names</a></li>
<li><a href="#intein">Intein containing coding region</a></li>
<li><a href="#partial_CDS">Partial coding regions in incomplete genomes</a></li>
<li><a href="#disrupted_genes">Gene fragments</a></li>
<li><a href="#introns">Intron Containing Genes</a></li>
<li><a href="#transpliced">Transpliced Genes</a></li>
<li><a href="#Split_genes">Split genes on two contigs</a></li>
<li><a href="#RNA">Ribosomal RNA, tRNA and other RNA features</a></li>
<li><a href="#Evidence_Qualifiers">Evidence Qualifiers</a></li>
<li><a href="#bacteriophage">Functional bacteriophage</a></li>
<li><a href="#transposons">Insertion sequences and transposons</a></li>
<li><a href="#db_xref">Data base cross references</a></li>
<li><a href="#GOterms">Gene Ontology</a></li>
<li><a href="#variation">Variation</a></li>
<li><a href="#Other">Other Annotation</a></li>
</ul>
</li>
</ol>
<h3 id="prepare-annotation-table">Prepare annotation table</h3>
<p>The features must be in a simple five-column tab-delimited table, called the feature table. The feature table specifies the location and type of each feature for tbl2asn or Sequin to include in the GenBank submission that is created. The first line of the table contains the following basic information:</p>
<pre><code>&gt;Features SeqId table_name
</code></pre><p>The SeqId must be the same as that used on the sequence. The table_name is optional. Subsequent lines of the table list the features. Columns are separated by tabs.</p>
<ul>
<li>Column 1: Start location of feature</li>
<li>Column 2: Stop location of feature</li>
<li>Column 3: Feature key</li>
<li>Column 4: Qualifier key</li>
<li>Column 5: Qualifier value</li>
</ul>
<p><a href="~/genomesubmit-Examples#fig2">Figure 2</a> shows a sample feature table and illustrates a number of points about the feature table format. The GenBank flatfile corresponding to this table is shown in <a href="~/genomesubmit-Examples#fig3">Figure 3</a></p>
<p>Features that are on the complementary strand (such as the gene abrB in the examples below) and its corresponding CDS, are indicated by reversing the interval locations. Please avoid unnecessary capitalization in all text entered in your table.</p>
<h4 id="gene-features">Gene features</h4>
<p>Gene features are usually a single interval, and their location should cover the intervals of all the relevant features such as promoters and operator binding sites. Gene names must follow the standard bacterial nomenclature rules of three lower case letters. Different loci are distinguished by a suffix of uppercase letters.</p>
<h4 id="example">Example</h4>
<pre><code>correct cytB
  incorrect CYTB
  incorrect cytochrome B
  incorrect orf1
  incorrect putative gene fragment
</code></pre><p>If a gene is a pseudogene, please do not add the word &quot;pseudo&quot; to the gene name or protein name. Instead use the /pseudo qualifier on the gene feature. Please see <a href="#disrupted_genes">Gene fragments</a> for more details.</p>
<h4 id="locustag">locus_tag</h4>
<p>All genes must be assigned a systematic gene identifier which must receive the locus_tag qualifier on the gene feature in the annotation table. Genes may also have functional names as assigned in the scientific literature. In this example, OBB_0001 is the systematic gene identifier, while abcD is the functional gene name. We recommend having the BioProject registration process auto-assign a locus_tag prefix, as they are not meant to confer meaning. The locus_tag prefix must be 3-12 alphanumeric characters and the first character may not be a digit. Additionally locus_tag prefixes are case sensitive. All chromosomes and plasmids of an individual genome must use the exactly same locus_tag prefix followed by an underscore and then an alphanumeric identification number that is unique within the given genome. Other than the single underscore used to separate the prefix from the identification number no other special characters can be used in the locus_tag. Locus_tags must only be used in combination with a gene feature. Read more about <a href="www.ncbi.nlm.nih.gov/genomes/locustag/Proposal.pdf">locus_tags</a> and their intended usage.</p>
<p>The use of locus_tag is supported in Sequin version 4.35 or newer. If you have an older version of Sequin please download the <a href="www.ncbi.nlm.nih.gov/Sequin">current version</a> .</p>
<h4 id="table-view-of-gene-with-both-biological-name-and-locustag">Table view of gene with both biological name and locus_tag</h4>
<pre><code>1 1575   gene
                gene    abcD
                locus_tag     OBB_0001
</code></pre><h4 id="flatfile-view">Flatfile view</h4>
<pre><code>gene     1..1575
        /gene=&quot;abcD&quot;
        /locus_tag=&quot;OBB_0001&quot;
</code></pre><h4 id="table-view-of-gene-with-only-locustag">Table view of gene with only locus_tag:</h4>
<pre><code>1 1575   gene
                locus_tag     OBB_0001
</code></pre><h4 id="flatfile-view">Flatfile view:</h4>
<pre><code>gene     1..157
        /locus_tag=&quot;OBB_0001&quot;
</code></pre><h4 id="proteinid">protein_id</h4>
<p>The submitter must assign an identification number to all proteins. NCBI uses this number to track proteins when sequences are updated. This number is indicated in the table by the CDS qualifier protein_id, and must have the format gnl|dbname|string, where dbname is a version of your lab name that you think will be unique (eg SmithUCSD), and string is the unique protein SeqID assigned by the submitter.</p>
<h4 id="example">Example</h4>
<p>The protein_id for abcD is <code>gnl|dbname|OBB_0001</code>. This identifier is saved with the record (in ASN.1 format), but it is not visible in the flatfile. We recommend using the locus_tag number as the protein identification number.</p>
<pre><code>1  1575    gene
                        gene    abcD
                        locus_tag     OBB_0001
1       1575    CDS
                        product AbcD
                        protein_id      gnl|SmithUCSD|OBB_0001
</code></pre><p>The protein_id is used for internal tracking in our database, it is important that the complete protein_id (dbname + string) not be duplicated by a genome center. Note that when WGS submissions are processed, the dbname in the protein_id is automatically changed to &apos;WGS:XXXX&apos;, where XXXX is the project&apos;s accession number prefix. After your genome is released into GenBank, the proteins are assigned accession numbers. We will provide a table of the protein SeqIDs and accession numbers for you to use in future <a href="~/genomesubmit#updating">updates</a> .</p>
<h4 id="protein-names">protein names</h4>
<p>All CDS features must have a product qualifier (protein name). NCBI protein naming conventions are adopted in part from the <a href="www.uniprot.org/docs/proknameprot">UniProt-Swiss-Prot Protein Knowledgebase</a></p>
<p>Consistent nomenclature is indispensable for communication, literature searching and data retrieval. Many species-specific communities have established gene nomenclature committees that try to assign consistent and, if possible, meaningful gene symbols. Other scientific communities have established protein nomenclatures for a set of proteins based on sequence similarity and/or function. But there is no established organization involved in the standardization of protein names, nor are there any efforts to establish naming rules that are valid across the largest spectrum of species possible.</p>
<p>Ambiguities regarding gene/protein names are a major problem in the literature and it is even worse in the sequence databases which tend to propagate the confusion. For this reason, we ask that you follow some basic guidelines in naming your proteins. The protein naming guidelines are based on the premise that a good and stable recommended name for a protein is a name that is as neutral as possible.</p>
<p>Guidelines for naming proteins:</p>
<ul>
<li>If it exists, use the approved nomenclature.</li>
<li>Use a concise name, not a description or phrase.</li>
<li>Ideally the name should be unique and attributed to all orthologs.</li>
<li>The protein name should not contain specific characteristics of the protein, and it should not reflect the function of the protein, its subcellular location, its domain structure, its molecular weight or its species of origin. This information can be included in the note.</li>
<li>In cases where the protein name is not known use &quot;unknown&quot; or &quot;hypothetical protein&quot; as the product name. We recommend the use of &quot;hypothetical protein&quot; as this will allow the locus_tag identifier to be appended to the product name in BLAST and Entrez summary lines.</li>
<li>Protein names may be denoted by the same symbol as the corresponding gene, but the symbol begins with a capital letter.</li>
<li>Avoid the use of molecular weights in protein names &quot;unicornase subunit A&quot; is preferred to &quot;unicornase 52 kDa subunit&quot;</li>
<li>Do not use the term &quot;homolog&quot; in a protein as this infers an evolutionary relationship that has generally not been determined.</li>
<li>Where possible, avoid the use of commas in protein names</li>
<li>Use lowercase letters, except when uppercase are required (for example, in acronyms such as DNA or ATP).</li>
<li>Wherever appropriate, the name should use American spelling conventions.</li>
<li>Avoid the use of Roman numerals where possible. Use instead Arabic numbers.</li>
<li>Do not build molecular weights into abbreviations</li>
<li>For proteins that belong to a multigene family, it is recommended that you choose a coherent nomenclature with numbers to specify the different members of the family.</li>
<li>When naming proteins which can be grouped into a family based on homology or according to a notion of shared function, the different members should be enumerated with a dash &quot;-&quot; followed by an Arabic number. e.g. &quot;desmoglein-1&quot;, &quot;desmoglein-2&quot;, etc.</li>
<li>Greek letters must be written in full e.g. &quot;alpha&quot;, and written entirely in lower case with the exception of &quot;Delta&quot; in the context of steroid/fatty acid metabolism nomenclature. Additionally the Greek letters that are followed by a number should be preceded or followed by a dash &quot;-&quot; e.g. &quot;unicornase alpha-1&quot;.</li>
<li>Do not use diacritics, such as accents, umlauts. Many computer sytems (ours included) can only understand ASCII characters.</li>
<li>Do not use plurals in a protein name. e.g. &quot;ankyrin repeats-containing protein 8&quot; is wrong.</li>
<li>Proteins of unknown function which contain a defined domain or motif, can be named according to the domain present. The name should be of the following type: &quot;<domain|repeat>-containing protein&quot;. e.g. &quot;PAS domain-containing protein 5&quot;.</domain|repeat></li>
</ul>
<p>Here are some examples of good protein names. These names all concisely describe the function of the protein, where known, and avoid references to structure, homology and species:</p>
<ul>
<li><code>cytochrome b</code></li>
<li><code>CytB</code></li>
<li><code>aconitate hydrase B</code></li>
<li><code>hypothetical protein</code></li>
<li><code>cytochrome b-like protein</code></li>
<li><code>4Fe-4S cluster binding protein</code></li>
<li><code>adenylyltransferase/ADP-heptose synthase</code></li>
<li><code>2-hydroxyhepta-2,4-diene-1,7-dioate isomerase</code></li>
<li><code>short-chain specific acyl-CoA dehydrogenase</code></li>
<li><code>formylmethanofuran--tetrahydromethanopterin formyltransferase</code></li>
<li><code>serine/threonine-protein kinase</code></li>
<li><code>translation initiation factor 1</code></li>
<li><code>triphosphoribosyl-dephospho-CoA synthetase</code></li>
<li><code>thiamine biosynthesis protein ThiC</code></li>
<li><code>PAS domain-containing protein 5</code></li>
<li><code>ABC transporter ATP-binding protein AlbC</code></li>
<li><code>stage 0 sporulation protein J</code></li>
</ul>
<p>Here are some examples of bad protein names:</p>
<ul>
<li><code>required for the efficient incorporation of molybdate into molybdoproteins</code><ul>
<li><em>This describes the protein&apos;s role in a biosynthetic process but is not a protein name.</em></li>
</ul>
</li>
</ul>
<ul>
<li><code>chaperone Hsp70; DNA biosynthesis; autoregulated heat shock proteins</code><ul>
<li><em>The name &quot;chaperone Hsp70&quot; is fine however the remaining comments would be best fielded as a note or in the function qualifier.</em></li>
</ul>
</li>
</ul>
<ul>
<li><code>putative carbonic anhdrase (EC 4.2.1.1)</code><ul>
<li><em>The EC number should not be part of the protein name but instead fielded in the EC_number qualifier</em></li>
</ul>
</li>
</ul>
<ul>
<li><code>similar to aconitrate hydrase B</code><ul>
<li><em>This statement is fine as a note, however as a protein name aconitrate hydrase B-like protein is preferred</em></li>
</ul>
</li>
</ul>
<ul>
<li><code>related to protein of unknown function</code><ul>
<li><em>uninformative name</em></li>
</ul>
</li>
</ul>
<ul>
<li><code>cytochrome b-like</code><ul>
<li><em>cytochrome b-like protein is preferred</em></li>
</ul>
</li>
</ul>
<ul>
<li><code>ABC transporter related</code><ul>
<li><em>vague name, there are many ABC transporters and subunits, be more specific</em></li>
</ul>
</li>
</ul>
<ul>
<li><code>pirin, N-terminal:pirin, C-terminal</code><ul>
<li><em>uniformative name noting similarity in N and C terminus</em></li>
</ul>
</li>
</ul>
<ul>
<li><code>helix-turn-helix motif</code><ul>
<li><em>Describes a motif or structural domain but is not an appropriate protein name.</em></li>
</ul>
</li>
</ul>
<ul>
<li><code>PP-loop</code><ul>
<li><em>Describes a motif or structural domain but is not an appropriate protein name.</em></li>
</ul>
</li>
</ul>
<ul>
<li><code>alpha/beta hydrolast fold</code><ul>
<li><em>Describes a motif or structural domain but is not an appropriate protein name.</em></li>
</ul>
</li>
</ul>
<ul>
<li><code>pentapeptide repeat</code><ul>
<li><em>Describes a motif or structural domain but is not an appropriate protein name.</em></li>
</ul>
</li>
</ul>
<ul>
<li><code>phosphopantetheine-binding domain</code><ul>
<li><em>Describes a motif or structural domain but is not an appropriate protein name.</em></li>
</ul>
</li>
</ul>
<ul>
<li><code>protein of unknown function:conserved</code><ul>
<li><em>uninformative name</em></li>
</ul>
</li>
</ul>
<ul>
<li><code>hypothetical 32.5 kDa protein homologous to phytoene and squalene synthethases</code><ul>
<li><em>Hypothetical protein alone is appropriate. The remaining comments should be fielded as a note.</em></li>
</ul>
</li>
</ul>
<ul>
<li><code>ribosomal protein L3 (E. coli)</code><ul>
<li><em>Protein names should not contain references to organism names. Ribosomal protein L3 is an appropriate name by itself.</em></li>
</ul>
</li>
</ul>
<ul>
<li><code>saccharopine dehydrogenase or related protein</code><ul>
<li><em>&quot;saccharopine dehydrogenase&quot; or &quot;saccharopine dehydrogenase-like protein&quot; would be more appropriate</em></li>
</ul>
</li>
</ul>
<ul>
<li><code>tyrosine-protein kinase (capsular polysaccharide biosynthesis)</code><ul>
<li><em>tyrosine-protein kinase is fine as a protein name but capsular polysaccharide biosynthesis would be more appropriate as a function.</em></li>
</ul>
</li>
</ul>
<ul>
<li><code>RimM protein, required for 16S rRNA processing</code><ul>
<li><em>RimM is fine as a protein name but descriptive comments should be placed in the note.</em></li>
</ul>
</li>
</ul>
<ul>
<li><code>involved in flagellar biosynthesis</code><ul>
<li><em>This is a functional comment and not a protein name.</em></li>
</ul>
</li>
</ul>
<h4 id="notes">Notes</h4>
<p>Please avoid including notes indicating a specific percentage of similarity to other entries in the database, since the corresponding record that you have pointed to may change and make your current note inaccurate, incorrect and obsolete. Descriptions, notes describing similarity to other proteins, and functional comments must be placed in the appropriate CDS qualifiers such as <code>note</code>, or <code>prot_desc</code>, as they are descriptors of the product. E.C. numbers must be fielded in an <code>EC_number</code> qualifier.</p>
<pre><code>start    stop    CDS
                        product DNA gyrase subunit B
                        EC_number     5.99.1.3
                        note    required for the gyration of DNA
</code></pre><p>Qualifiers that can be used on the CDS feature are:</p>
<pre><code>start   stop    CDS
                        product
                        prot_desc
                        function
                        EC_number
                        note
                        experiment
                        inference
                        go_component
                        go_process
                        go_function
                        db_xref
                        pseudo
                        exception
                        transl_except
</code></pre><h4 id="bifunctional-proteins">Bifunctional Proteins</h4>
<p>If a protein contains two separate and distinct functions or if it has more than one name, it can be annotated in several ways as outlined below.</p>
<p>Table view:</p>
<pre><code>start  stop    CDS
                        product adenylyltransferase/ADP-heptose synthase
                        note    bifunctional
                        EC_number     2.7.7.2
                        EC_number     1.4.1.13
</code></pre><p>or</p>
<pre><code>start    stop    CDS
                        product bifunctional adenylyltransferase/ADP-heptose synthase cyclohydrolase
                        EC_number     2.7.7.2
                        EC_number     1.4.1.13
</code></pre><p>or</p>
<pre><code>start    stop    CDS
                        product FolD 
                        function        adenylyltransferase
                        function        ADP-heptose synthase cyclohydrolase
                        note    bifunctional
                        EC_number     2.7.7.2
                        EC_number     1.4.1.13
</code></pre><h4 id="intein-containing-coding-regions">Intein-containing coding regions</h4>
<p>Intein-containing coding regions must be represented as follows:</p>
<pre><code>946506  950039  gene
                        gene    recA
                        locus_tag     OBB_0010
946506  950039  CDS
                        product DNA recombination protein precursor
                        protein_id      gnl|dbname|OBB_0010
946506  946790  misc_feature
948057  950036
                        note    DNA recombination protein
946791  948056  misc_feature
                        note    intein
</code></pre><p>Inteins should be annotated with two <code>mat_peptide</code> features, one for the intein and one for the final protein. We also add <code>precursor</code> to the product name on the CDS feature. Unfortunately, you can not add a mat_peptide feature in a table. Instead, you can add a misc_feature and we can convert them for you. Please see accession number <a href="www.ncbi.nlm.nih.gov/nuccore/AY847267">AY847267</a> for an example of an intein-containing protein.</p>
<h4 id="partial-coding-regions-in-incomplete-genomes">Partial coding regions in incomplete genomes</h4>
<p>Annotate a partial coding region using the <code>&lt;</code> or <code>&gt;</code> in your feature table to designate the feature as either 5&apos; or 3&apos; partial. Although coding region must begin at the first nucleotide present, the translation starts at the first complete codon.</p>
<h4 id="example">Example</h4>
<p>In the first example below, the <code>&lt;</code> designates this coding region as 5&apos; partial and <code>codon_start 3</code> tells the software to start translation with the third nucleotide of the CDS. Note that if the <code>codon_start</code> is not specified, then the software assumes a <code>codon_start</code> of <code>1</code>. The second coding region below is partial at the 3&apos; end so <code>&gt;</code> is used to indicate a 3&apos; partial feature. The third example is of a 3&apos; partial coding region on the complementary or minus strand.</p>
<pre><code>&lt;1   497     gene
                        gene    abcD
                        locus_tag     OBB_0001
&lt;1   497     CDS
                        product AbcD
                        note    similar to Bacillus subtilis aldolase
                        codon_start     3
                        protein_id      gnl|dbname|OBB_0001
200     &gt;1575        gene
                        gene    xyzA
                        locus_tag     OBB_0002
200     &gt;1575        CDS
                        product actin-like protein
                        protein_id      gnl|dbname|OBB_0002
436     &gt;1   gene
                        gene    nirK
                        locus_tag     OBB_0003
436     &gt;1   CDS
                        product NirK
                        protein_id      gnl|dbname|OBB_0003
</code></pre><p>Here are <a href="~/examples.wgs#partialcds">more examples of formatting partial CDS features</a> .</p>
<p>Partial coding regions may be used for incomplete genomes only. All coding regions annotated on finished genomes must be complete at both the 5&apos; and 3&apos; ends.</p>
<h4 id="disrupted-genes-and-gene-fragments">Disrupted genes and gene fragments</h4>
<p>Sometimes a genome will have adjacent or nearby genes that seem to be only part of a protein. In many cases these indicate a possible problem with the sequence and/or annotation. A related issue is the presence of internal stop codons in the conceptual translation of a CDS that looks like it should be a real CDS. These problems may be due to a variety of reasons, including mutations or sequencing artifacts. They can be annotated in a number of ways:</p>
<ol>
<li><p><em>Annotate the gene as a pseudogene.</em> If multiple gene fragments were present initially, then add a single gene feature which covers all of the potential coding regions and add the pseudo qualifier indicating this is a pseudogene. If known, a note qualifier may be added indicating why this gene is disrupted.</p>
<pre><code>1        200     gene
                        gene    phoA
                        gene_desc     alkaline phosphatase
                        locus_tag     OBB_0001
                        pseudo
                        note    frameshift
</code></pre></li>
<li><p><em>Alternatively, if you are not sure if the disrupted gene is a &quot;pseudogene&quot; you can just use a gene feature without the <code>/pseudo</code>.</em> Please use the complete nucleotide spans of the frameshifted gene. A note can be added to indicate the reason for the incomplete translation.</p>
<pre><code>1    200     gene
                        gene    phoA
                        gene_desc     alkaline phosphatase
                        locus_tag     OBB_0001
                        note    nonfunctional due to frameshift
</code></pre></li>
<li><p><em>A coding region containing a frameshift that is thought to be corrected by ribosomal slippage can be annotated using joined feature spans.</em> Joined spans on a feature are used to combine two non-contiguous regions of sequence that are joined together to encode a protein, for example. This is typically used to combine eukaryotic exons to translate the coding region. To create a join CDS you must specify the spans of each contiguous region of sequence that encodes the protein. The use of the joined feature spans is rare in bacteria.</p>
<pre><code>333255  333181  CDS
333179  332157
                        product AbcD
                        protein_id     gnl|dbname|OBB_0001
                        exception      ribosomal slippage
</code></pre><p>In this case the CDS must also include an exception qualifier with the exact text <code>ribosomal slippage</code>. If you include a join feature for a different reason, please include a <code>note</code> qualifier indicating why the two nucleotide spans are joined.</p>
</li>
<li><p><em>If a gene is localized but the translation is unknown, it is possible to simply annotate the gene feature without a corresponding coding region.</em> The <code>gene_desc</code> qualifier can be used to annotate both the gene symbol and the gene description.</p>
<pre><code>1   200     gene
                        gene    phoA
                        gene_desc     alkaline phosphatase
                        locus_tag     OBB_0001
</code></pre></li>
<li><p><em>A gene containing an authentic frameshift induced by phase variation can be represented by a gene feature with a <code>note</code>.</em></p>
<pre><code>1    200     gene
                        gene    phoA
                        gene_desc     alkaline phosphatase
                        locus_tag     OBB_0001
                        note    authentic frameshift induced by phase variation; This region contains 
                        an authentic frameshift or in-frame stop in the coding sequence and is not the 
                        result of a sequencing error
</code></pre></li>
</ol>
<h4 id="intron-containing-genes">Intron-Containing Genes</h4>
<p>While rare, there are some examples of bacterial genes containing introns. Annotate the gene feature of any intron-containing gene such that the gene feature spans are a single span covering all exons and introns. The actual feature (CDS, tRNA, etc.) should then be annotated with sets of nucleotide spans showing how the exons are joined to create the correct product. In this example there are two exons transcribed to create a tRNA. The first exon is from 1456 to 1419 and the second is from 1400 to 1361. Note how the gene feature spans encompass both exons and the intron.</p>
<pre><code>1456    1361    gene
                        locus_tag     APO_t01
1456    1419    tRNA
1400    1361
                        product tRNA-Cys
</code></pre><h4 id="transpliced-genes">Transpliced Genes</h4>
<p>Transpliced genes are the exception to the rule for annotating gene feature spans. Transpliced genes are similar to intron containing genes except the two pieces of the gene are found on different regions of the chromosome. These genes are transcribed as two or more separate RNA products that are transpliced into a single mRNA or tRNA. To annotate this using a table, enter the nucleotide spans so that the complementary (minus strand) spans are arranged from high to low and vice versa for the plus strand.</p>
<pre><code>36700   36618   gene
86988   87064
                        locus_tag    NEQ_t38
                        exception    trans-splicing
36631   36618   misc_feature
                        note    sequence cleaved during processing of trans-spliced tRNAs
36673   36635
87030   87064   tRNA
                        product tRNA-Glu
                        exception    trans-splicing
                        note    this trans-spliced tRNA consists of two halves on mixed strands; it shares a 3&apos; half with another tRNA
</code></pre><p>Flatfile view:</p>
<pre><code>gene            join(complement(36618..36700),86988..87064)
                     /locus_tag=&quot;NEQ_t38&quot;
                     /trans_splicing
     misc_feature    complement(36618..36631)
                     /locus_tag=&quot;NEQ_t38&quot;
                     /note=&quot;sequence cleaved during processing of trans-spliced tRNAs&quot;
     tRNA            join(complement(36635..36673),87030..87064)
                     /locus_tag=&quot;NEQ_t38&quot;
                     /product=&quot;tRNA-Glu&quot;
                     /trans_splicing
                     /note=&quot;this trans-spliced tRNA consists of two halves on
                     mixed strands; it shares a 3&apos; half with another tRNA&quot;
</code></pre><h4 id="split-genes-on-two-contigs">Split genes on two contigs</h4>
<p>NEW (Sept 2012): Sometimes in incomplete genomes the ends of a gene may be on different contigs. When certain that the two pieces are part of the same gene, annotate these as separate genes with unique locus_tags, plus separate CDS with different protein_id&apos;s. In addition, link the features together with notes that refer to the other part of the gene. However, do not create extremely short features, for example if one end is only the start methinione or only a few amino acids before the stop codon.</p>
<h4 id="example">Example</h4>
<pre><code>&gt;Feature Cont01.00111
5000    &gt;7500        gene
                        locus_tag     KCS_2223A
5488    5500    CDS
6000    &gt;7200
                        product enolase
                        protein_id    gnl|dbname|KCS_2223A
                        note    5&apos; end; 3&apos; end is gene KCS_2223B on contig Cont01.00224
</code></pre><pre><code>&gt;Feature Cont01.00224
&lt;1   1000    gene    
                        locus_tag     KCS_2223B
&lt;100 876     CDS
                        product enolase
                        protein_id    gnl|dbname|KCS_2223B
                        note    3&apos; end; 5&apos; end is gene KCS_2223A on contig Cont01.00111
</code></pre><h4 id="ribosomal-rna-trna-and-other-rna-features">Ribosomal RNA, tRNA and other RNA features</h4>
<p>RNA features (rRNA, tRNA, ncRNA) must include a corresponding gene feature with a locus_tag qualifier. Please be sure to specify which amino acid the tRNA gene corresponds to. If the amino acid of a tRNA is unknown, use tRNA-Xxx as the product, as in the example. Many submitters like to label the tRNAs such as tRNA-Gly1, etc. If you wish to do this please include &quot;tRNA-Gly1&quot; as a note and not in /gene. The use of /gene is reserved for the actual biological gene symbol such as &quot;trnG&quot;. If a tRNA is a pseudogene, please use the /pseudo qualifier.</p>
<p>Annotate ncRNAs that belong to one of the INSDC <a href="www.insdc.org/documents/ncrna-vocabulary">ncRNA_class</a> as an ncRNA feature, with the appropriate value in the required /ncRNA_class qualifier. Regions of an RNA should be annotated as a misc_feature (eg, leader sequences), or a misc_binding feature if they bind a known molecule (eg, riboswitches). See <a href="#Other">Other Annotation</a> for examples of regions of RNAs.</p>
<p>Some rRNA, tRNA, ncRNA examples:</p>
<pre><code>&lt;1      400     gene
                        locus_tag     OBB_0001
&lt;1      400     rRNA
                        product 16S ribosomal RNA
401     500     gene
                        gene    trnG
                        note    tRNA-Gly1
                        locus_tag     OBB_0002
401     500     tRNA
                        product tRNA-Gly
501     600     gene
                        locus_tag     OBB_0003
501     600     tmRNA
                        product tmRNA
601     700     gene
                        locus_tag     OBB_0004
601     700     tRNA
                        product tRNA-Xxx
701     800     gene
                        locus_tag     OBB_0005
                        pseudo
701     800     tRNA    
                        product tRNA-Phe
                        pseudo
801     900     gene
                        locus_tag       OBB_0006
801     900     ncRNA
                        ncRNA_class     SRP_RNA
                        product RNA component of signal recognition particle
</code></pre><h4 id="evidence-qualifiers">Evidence Qualifiers</h4>
<p>At the 2005 annual meeting of the International Nucleotide Sequence Databases (INSD), DDBJ, EMBL and GenBank agreed to adopt two qualifiers to describe the evidence for features in sequence records. These are &quot;/experimental=text&quot; and &quot;/inference=TYPE:text&quot;, where &apos;TYPE&apos; is from a select list and &apos;text&apos; is structured text. These new qualifiers replace &quot;evidence=experimental&quot; and &quot;evidence=non-experimental&quot;, respectively, which are no longer supported. Read more about <a href="~/evidence">Evidence Qualifiers</a></p>
<pre><code>1     100   gene  
                locus_tag   Test_0001
1     100   CDS
                product     RecA
                protein_id  gnl|center_name|Test_0001
                inference   ab initio prediction:Genscan:2.0
200   300   gene
                locus_tag   Test_0002
200   300   CDS
                product     SecA
                protein_id  gnl|center_name|Test_0002
                inference   similar to DNA sequence, (same species):INSD:DQ060639.1
400   500   gene  
                locus_tag   Test_0003
400   500   CDS
                product     ribonuclease R
                protein_id  gnl|center_name|Test_0003
                inference   protein motif:InterPro:IPR001900
                db_xref InterPro:IPR001900
600   700   gene  
                locus_tag   Test_0004
600   700   CDS
                product     nitroreductase A
                protein_id  gnl|center_name|Test_0004
                experiment  expression of GST fusion protein
</code></pre><h4 id="functional-bacteriophage">Functional bacteriophage</h4>
<p>If a bacterial genome contains a functional phage, an additional source feature must be included with the spans covering the complete phage sequence. However, if the phage is not functional or if you are not sure, annotate it as a misc_feature.</p>
<h4 id="example">Example</h4>
<pre><code>361      4200    source
                        organism        Bacteriophage xyz
</code></pre><h4 id="insertion-sequences-and-transposons">Insertion sequences and transposons</h4>
<p>Insertion sequences and transposons must be annotated as repeat_region features. The name of the insertion sequence or transposon must be added in a insertion_seq or transposon qualifier. Note that transposons and insertion sequences should not be given locus_tags.</p>
<pre><code>1     100     repeat_region
                        mobile_element  insertion sequence:IS1363

500   600     repeat_region
                        mobile_element  transposon:Athena-Av1
</code></pre><h4 id="data-base-cross-references">Data base cross references</h4>
<p>A variety of data base cross references can be added to a feature. These appear as /db_xref on the features. This qualifier serves as a vehicle for linking of sequence records to other external databases. See the full list of <a href="~/collab/db_xref">db_xref</a> .</p>
<pre><code>1     100   gene  
                locus_tag   Test_0001
1     100   CDS
                product     RecA
                protein_id  gnl|center_name|Test_0001
                db_xref InterPro:IPR000111

180     210   misc_feature
                note     yybP-ykoY element
                db_xref RFAM:RF00080
</code></pre><h4 id="gene-ontology">Gene Ontology</h4>
<p>GO (Gene Ontology) terms can be included in genomes in order to describe protein functionality. Gene Ontology (GO) terms can be indicated with the following qualifiers</p>
<pre><code>1       100     CDS
                        product AbcD
                        go_component    exocyst|0000145
                        go_process      regulation of transcription, DNA-dependent|0006355
                        go_process      exocytosis|0006887
                        go_function     DNA binding|0003677
</code></pre><p>The value field is separated by vertical bars &apos;|&apos; into a descriptive string, the GO identifier (leading zeroes are retained), and optionally a PubMed ID and one or more evidence codes. The evidence code is the fourth token, so include blank fields, as necessary (eg the last qualifier has no PubMed ID so the third field is blank). See examples on the <a href="~/eukaryotic_genome_submission_annotation#GOterms">detailed eukaryotic annotation</a> page.  </p>
<h4 id="variation">Variation</h4>
<p>Polymorphisms in the sequence can be shown with variation features. Include one of the polymorphisms in the sequence (usually, this is the most commonly seen sequence), and then add a variation feature in the .tbl file for each of the other possibilities.</p>
<ul>
<li>The <code>variation</code> feature requires a <code>replace</code> qualifier, whose value is the sequence of the polymorphism that is NOT in the submitted sequence. For example, if <code>CCC</code> is most common at position 100-102 but there is also <code>CC</code> (a substitution), <code>CCCCC</code> (an insertion), and nothing (a deletion) then the sequence will have <code>CCC</code> at that location and you would include three variation features, one for each polymorphism.</li>
<li>For an insertion polymorphism, a carat (<code>^</code>) is part of the start location.</li>
<li>When the polymorphism is a complete deletion, then the replace value is just two double-quotes.</li>
<li>You can also include optional qualifiers: <code>note</code>, and the <code>frequency</code> with which the other sequence is found.</li>
</ul>
<p>Here is an example with all of those options:</p>
<pre><code>100    102    variation
            replace    cc
            note    polymorphism
100^    102    variation
            replace    ccccc
            frequency    0.1
100    102    variation
            replace    &quot;&quot;
            note    deletion
</code></pre><p>Those features will appear like this in the GenBank view:</p>
<pre><code>variation       100..102
                     /note=&quot;polymorphism&quot;
                     /replace=&quot;cc&quot;
     variation       100^102
                     /frequency=&quot;0.1&quot;
                     /replace=&quot;ccccc&quot;
     variation       100..102
                     /note=&quot;deletion&quot;
                     /replace=&quot;&quot;
</code></pre><h4 id="other-annotation">Other Annotation</h4>
<p>Riboswitches should be annotated using the <code>misc_binding</code> feature if the bound moiety is known, for example:</p>
<pre><code>1     100    misc_binding
                        note cobalamin riboswitch
                        bound_moiety adenosylcobalamin
</code></pre><p>If the bound moiety is unknown or if the sequence is a leader sequence, annotate as a misc_feature, for example:</p>
<pre><code>1     100    misc_feature
                        note yybP-ykoY element
</code></pre><p><code>misc_feature</code> and <code>misc_binding</code> features do not have an associated gene feature. If it is desired to tag these features with a locus_tag-like identifier, then include that value in the note, separated from other information by a semi-colon and space.</p>

          

        </div>
      </div>
    

    <div id='shared-content-1' nid='1092'>
      <div class='rightnav'>
        <h2>GenBank Resources</h2>
        <ul>
          <li>
            <a href="/~/">GenBank Home</a></li>
          <li>
            <a href="/~/submit_types">Submission Types</a></li>
          <li>
            <a href="/~/submit">Submission Tools</a></li>
          <li>
            <a href="http://www.ncbi.nlm.nih.gov/nuccore/">Search GenBank</a></li>
          <li>
            <a href="/~/update">Update GenBank Records</a></li>
        </ul>
      </div>
    </div>

  </body>
</html>

