

<html>
  <head>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta charset="UTF-8" />
    <title></title>
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <title>GenBank Submission Tools</title>
  </head>

  <body>

    
      <div class='node clear-block'>
        <div class='content'>

          
            <h1 id="sample-genbank-record">Sample GenBank Record</h1>
<style type="text/css">#maincontent h2 { padding-top: 1em; } pre { font-family: Courier, Monospace; border: solid 1px #ccc; background-color: #ffe; padding: 1em; margin: 1.2em; font-size: 1.0em; } code { font-weight: bold; }</style>

<p>This page presents an annotated sample GenBank record (accession number <code>U49845</code>) in its <em>GenBank Flat File</em> format. You can see the corresponding <a href="nucleotide/U49845">live record for U49845</a>, and see <a href="#OtherFeaturesB">examples of other records</a> that show a range of biological features.</p>
<pre><code>[LOCUS](#LocusB)       [SCU49845](#LocusNameB)     [5028 bp](#SequenceLengthB)    [DNA](#MoleculeTypeB)             [PLN](#GenBankDivisionB)       [21-JUN-1999](#ModificationDateB)
[DEFINITION](#DefinitionB)  Saccharomyces cerevisiae TCP1-beta gene, partial cds, and Axl2p
            (AXL2) and Rev7p (REV7) genes, complete cds.
[ACCESSION](#AccessionB)   U49845
[VERSION](#VersionB)     U49845.1  [GI](#GInB):1293613
[KEYWORDS](#KeywordsB)    .
[SOURCE](#SourceB)      Saccharomyces cerevisiae (baker&apos;s yeast)
  [ORGANISM](#OrganismB)  Saccharomyces cerevisiae
            Eukaryota; Fungi; Ascomycota; Saccharomycotina; Saccharomycetes;
            Saccharomycetales; Saccharomycetaceae; Saccharomyces.
[REFERENCE](#ReferenceB)   1  (bases 1 to 5028)
  [AUTHORS](#AuthorsB)   Torpey,L.E., Gibbs,P.E., Nelson,J. and Lawrence,C.W.
  [TITLE](#TitleB)     Cloning and sequence of REV7, a gene whose function is required for
            DNA damage-induced mutagenesis in Saccharomyces cerevisiae
  [JOURNAL](#JournalB)   Yeast 10 (11), 1503-1509 (1994)
  [PUBMED](#PubMedB)    7871890
REFERENCE   2  (bases 1 to 5028)
  AUTHORS   Roemer,T., Madden,K., Chang,J. and Snyder,M.
  TITLE     Selection of axial growth sites in yeast requires Axl2p, a novel
            plasma membrane glycoprotein
  JOURNAL   Genes Dev. 10 (7), 777-793 (1996)
  PUBMED    8846915
REFERENCE   3  (bases 1 to 5028)
  AUTHORS   Roemer,T.
  TITLE     [Direct Submission](#SubmitterBlockB)
  JOURNAL   Submitted (22-FEB-1996) Terry Roemer, Biology, Yale University, New
            Haven, CT, USA
[FEATURES](#FeaturesB)             Location/Qualifiers
     [source](#FeaturesSourceB)          1..5028
                     /organism=&quot;Saccharomyces cerevisiae&quot;
                     /[db_xref=&quot;taxon:4932&quot;](#TaxonB)
                     /chromosome=&quot;IX&quot;
                     /map=&quot;9&quot;
     [CDS](#CDSB)             [&lt;1..206](#BaseSpanB)
                     /codon_start=3
                     /product=&quot;TCP1-beta&quot;
                     [/protein_id](#ProteinIDB)=&quot;AAA98665.1&quot;
                     /db_xref=&quot;[GI](#GIpB):1293614&quot;
                     [/translation](#TranslationB)=&quot;SSIYNGISTSGLDLNNGTIADMRQLGIVESYKLKRAVVSSASEA
                     AEVLLRVDNIIRARPRTANRQHM&quot;
     [gene](#GeneB)            687..3158
                     /gene=&quot;AXL2&quot;
     CDS             687..3158
                     /gene=&quot;AXL2&quot;
                     /note=&quot;plasma membrane glycoprotein&quot;
                     /codon_start=1
                     /function=&quot;required for axial budding pattern of S.
                     cerevisiae&quot;
                     /product=&quot;Axl2p&quot;
                     [/protein_id](#ProteinIDB)=&quot;AAA98666.1&quot;
                     /db_xref=&quot;[GI](#GIpB):1293615&quot;
                     [/translation](#TranslationB)=&quot;MTQLQISLLLTATISLLHLVVATPYEAYPIGKQYPPVARVNESF
                     TFQISNDTYKSSVDKTAQITYNCFDLPSWLSFDSSSRTFSGEPSSDLLSDANTTLYFN
                     VILEGTDSADSTSLNNTYQFVVTNRPSISLSSDFNLLALLKNYGYTNGKNALKLDPNE
                     VFNVTFDRSMFTNEESIVSYYGRSQLYNAPLPNWLFFDSGELKFTGTAPVINSAIAPE
                     TSYSFVIIATDIEGFSAVEVEFELVIGAHQLTTSIQNSLIINVTDTGNVSYDLPLNYV
                     YLDDDPISSDKLGSINLLDAPDWVALDNATISGSVPDELLGKNSNPANFSVSIYDTYG
                     DVIYFNFEVVSTTDLFAISSLPNINATRGEWFSYYFLPSQFTDYVNTNVSLEFTNSSQ
                     DHDWVKFQSSNLTLAGEVPKNFDKLSLGLKANQGSQSQELYFNIIGMDSKITHSNHSA
                     NATSTRSSHHSTSTSSYTSSTYTAKISSTSAAATSSAPAALPAANKTSSHNKKAVAIA
                     CGVAIPLGVILVALICFLIFWRRRRENPDDENLPHAISGPDLNNPANKPNQENATPLN
                     NPFDDDASSYDDTSIARRLAALNTLKLDNHSATESDISSVDEKRDSLSGMNTYNDQFQ
                     SQSKEELLAKPPVQPPESPFFDPQNRSSSVYMDSEPAVNKSWRYTGNLSPVSDIVRDS
                     YGSQKTVDTEKLFDLEAPEKEKRTSRDVTMSSLDPWNSNISPSPVRKSVTPSPYNVTK
                     HRNRHLQNIQDSQSGKNGITPTTMSTSSSDDFVPVKDGENFCWVHSMEPDRRPSKKRL
                     VDFSNKSNVNVGQVKDIHGRIPEML&quot;
     gene            [complement](#ComplementB)(3300..4037)
                     /gene=&quot;REV7&quot;
     CDS             [complement](#ComplementB)(3300..4037)
                     /gene=&quot;REV7&quot;
                     /codon_start=1
                     /product=&quot;Rev7p&quot;
                     [/protein_id](#ProteinIDB)=&quot;AAA98667.1&quot;
                     /db_xref=&quot;[GI](#GIpB):1293616&quot;
                     [/translation](#TranslationB)=&quot;MNRWVEKWLRVYLKCYINLILFYRNVYPPQSFDYTTYQSFNLPQ
                     FVPINRHPALIDYIEELILDVLSKLTHVYRFSICIINKKNDLCIEKYVLDFSELQHVD
                     KDDQIITETEVFDEFRSSLNSLIMHLEKLPKVNDDTITFEAVINAIELELGHKLDRNR
                     RVDSLEEKAEIERDSNWVKCQEDENLPDNNGFQPPKIKLTSLVGSDVGPLIIHQFSEK
                     LISGDDKILNGVYSQYEEGESIFGSLF&quot;
[ORIGIN](#OriginB)
        1 gatcctccat atacaacggt atctccacct caggtttaga tctcaacaac ggaaccattg
       61 ccgacatgag acagttaggt atcgtcgaga gttacaagct aaaacgagca gtagtcagct
      121 ctgcatctga agccgctgaa gttctactaa gggtggataa catcatccgt gcaagaccaa
      181 gaaccgccaa tagacaacat atgtaacata tttaggatat acctcgaaaa taataaaccg
      241 ccacactgtc attattataa ttagaaacag aacgcaaaaa ttatccacta tataattcaa
      301 agacgcgaaa aaaaaagaac aacgcgtcat agaacttttg gcaattcgcg tcacaaataa
      361 attttggcaa cttatgtttc ctcttcgagc agtactcgag ccctgtctca agaatgtaat
      421 aatacccatc gtaggtatgg ttaaagatag catctccaca acctcaaagc tccttgccga
      481 gagtcgccct cctttgtcga gtaattttca cttttcatat gagaacttat tttcttattc
      541 tttactctca catcctgtag tgattgacac tgcaacagcc accatcacta gaagaacaga
      601 acaattactt aatagaaaaa ttatatcttc ctcgaaacga tttcctgctt ccaacatcta
      661 cgtatatcaa gaagcattca cttaccatga cacagcttca gatttcatta ttgctgacag
      721 ctactatatc actactccat ctagtagtgg ccacgcccta tgaggcatat cctatcggaa
      781 aacaataccc cccagtggca agagtcaatg aatcgtttac atttcaaatt tccaatgata
      841 cctataaatc gtctgtagac aagacagctc aaataacata caattgcttc gacttaccga
      901 gctggctttc gtttgactct agttctagaa cgttctcagg tgaaccttct tctgacttac
      961 tatctgatgc gaacaccacg ttgtatttca atgtaatact cgagggtacg gactctgccg
     1021 acagcacgtc tttgaacaat acataccaat ttgttgttac aaaccgtcca tccatctcgc
     1081 tatcgtcaga tttcaatcta ttggcgttgt taaaaaacta tggttatact aacggcaaaa
     1141 acgctctgaa actagatcct aatgaagtct tcaacgtgac ttttgaccgt tcaatgttca
     1201 ctaacgaaga atccattgtg tcgtattacg gacgttctca gttgtataat gcgccgttac
     1261 ccaattggct gttcttcgat tctggcgagt tgaagtttac tgggacggca ccggtgataa
     1321 actcggcgat tgctccagaa acaagctaca gttttgtcat catcgctaca gacattgaag
     1381 gattttctgc cgttgaggta gaattcgaat tagtcatcgg ggctcaccag ttaactacct
     1441 ctattcaaaa tagtttgata atcaacgtta ctgacacagg taacgtttca tatgacttac
     1501 ctctaaacta tgtttatctc gatgacgatc ctatttcttc tgataaattg ggttctataa
     1561 acttattgga tgctccagac tgggtggcat tagataatgc taccatttcc gggtctgtcc
     1621 cagatgaatt actcggtaag aactccaatc ctgccaattt ttctgtgtcc atttatgata
     1681 cttatggtga tgtgatttat ttcaacttcg aagttgtctc cacaacggat ttgtttgcca
     1741 ttagttctct tcccaatatt aacgctacaa ggggtgaatg gttctcctac tattttttgc
     1801 cttctcagtt tacagactac gtgaatacaa acgtttcatt agagtttact aattcaagcc
     1861 aagaccatga ctgggtgaaa ttccaatcat ctaatttaac attagctgga gaagtgccca
     1921 agaatttcga caagctttca ttaggtttga aagcgaacca aggttcacaa tctcaagagc
     1981 tatattttaa catcattggc atggattcaa agataactca ctcaaaccac agtgcgaatg
     2041 caacgtccac aagaagttct caccactcca cctcaacaag ttcttacaca tcttctactt
     2101 acactgcaaa aatttcttct acctccgctg ctgctacttc ttctgctcca gcagcgctgc
     2161 cagcagccaa taaaacttca tctcacaata aaaaagcagt agcaattgcg tgcggtgttg
     2221 ctatcccatt aggcgttatc ctagtagctc tcatttgctt cctaatattc tggagacgca
     2281 gaagggaaaa tccagacgat gaaaacttac cgcatgctat tagtggacct gatttgaata
     2341 atcctgcaaa taaaccaaat caagaaaacg ctacaccttt gaacaacccc tttgatgatg
     2401 atgcttcctc gtacgatgat acttcaatag caagaagatt ggctgctttg aacactttga
     2461 aattggataa ccactctgcc actgaatctg atatttccag cgtggatgaa aagagagatt
     2521 ctctatcagg tatgaataca tacaatgatc agttccaatc ccaaagtaaa gaagaattat
     2581 tagcaaaacc cccagtacag cctccagaga gcccgttctt tgacccacag aataggtctt
     2641 cttctgtgta tatggatagt gaaccagcag taaataaatc ctggcgatat actggcaacc
     2701 tgtcaccagt ctctgatatt gtcagagaca gttacggatc acaaaaaact gttgatacag
     2761 aaaaactttt cgatttagaa gcaccagaga aggaaaaacg tacgtcaagg gatgtcacta
     2821 tgtcttcact ggacccttgg aacagcaata ttagcccttc tcccgtaaga aaatcagtaa
     2881 caccatcacc atataacgta acgaagcatc gtaaccgcca cttacaaaat attcaagact
     2941 ctcaaagcgg taaaaacgga atcactccca caacaatgtc aacttcatct tctgacgatt
     3001 ttgttccggt taaagatggt gaaaattttt gctgggtcca tagcatggaa ccagacagaa
     3061 gaccaagtaa gaaaaggtta gtagattttt caaataagag taatgtcaat gttggtcaag
     3121 ttaaggacat tcacggacgc atcccagaaa tgctgtgatt atacgcaacg atattttgct
     3181 taattttatt ttcctgtttt attttttatt agtggtttac agatacccta tattttattt
     3241 agtttttata cttagagaca tttaatttta attccattct tcaaatttca tttttgcact
     3301 taaaacaaag atccaaaaat gctctcgccc tcttcatatt gagaatacac tccattcaaa
     3361 attttgtcgt caccgctgat taatttttca ctaaactgat gaataatcaa aggccccacg
     3421 tcagaaccga ctaaagaagt gagttttatt ttaggaggtt gaaaaccatt attgtctggt
     3481 aaattttcat cttcttgaca tttaacccag tttgaatccc tttcaatttc tgctttttcc
     3541 tccaaactat cgaccctcct gtttctgtcc aacttatgtc ctagttccaa ttcgatcgca
     3601 ttaataactg cttcaaatgt tattgtgtca tcgttgactt taggtaattt ctccaaatgc
     3661 ataatcaaac tatttaagga agatcggaat tcgtcgaaca cttcagtttc cgtaatgatc
     3721 tgatcgtctt tatccacatg ttgtaattca ctaaaatcta aaacgtattt ttcaatgcat
     3781 aaatcgttct ttttattaat aatgcagatg gaaaatctgt aaacgtgcgt taatttagaa
     3841 agaacatcca gtataagttc ttctatatag tcaattaaag caggatgcct attaatggga
     3901 acgaactgcg gcaagttgaa tgactggtaa gtagtgtagt cgaatgactg aggtgggtat
     3961 acatttctat aaaataaaat caaattaatg tagcatttta agtataccct cagccacttc
     4021 tctacccatc tattcataaa gctgacgcaa cgattactat tttttttttc ttcttggatc
     4081 tcagtcgtcg caaaaacgta taccttcttt ttccgacctt ttttttagct ttctggaaaa
     4141 gtttatatta gttaaacagg gtctagtctt agtgtgaaag ctagtggttt cgattgactg
     4201 atattaagaa agtggaaatt aaattagtag tgtagacgta tatgcatatg tatttctcgc
     4261 ctgtttatgt ttctacgtac ttttgattta tagcaagggg aaaagaaata catactattt
     4321 tttggtaaag gtgaaagcat aatgtaaaag ctagaataaa atggacgaaa taaagagagg
     4381 cttagttcat cttttttcca aaaagcaccc aatgataata actaaaatga aaaggatttg
     4441 ccatctgtca gcaacatcag ttgtgtgagc aataataaaa tcatcacctc cgttgccttt
     4501 agcgcgtttg tcgtttgtat cttccgtaat tttagtctta tcaatgggaa tcataaattt
     4561 tccaatgaat tagcaatttc gtccaattct ttttgagctt cttcatattt gctttggaat
     4621 tcttcgcact tcttttccca ttcatctctt tcttcttcca aagcaacgat ccttctaccc
     4681 atttgctcag agttcaaatc ggcctctttc agtttatcca ttgcttcctt cagtttggct
     4741 tcactgtctt ctagctgttg ttctagatcc tggtttttct tggtgtagtt ctcattatta
     4801 gatctcaagt tattggagtc ttcagccaat tgctttgtat cagacaattg actctctaac
     4861 ttctccactt cactgtcgag ttgctcgttt ttagcggaca aagatttaat ctcgttttct
     4921 ttttcagtgt tagattgctc taattctttg agctgttctc tcagctcctc atatttttct
     4981 tgccatgact cagattctaa ttttaagcta ttcaatttct ctttgatc
//
</code></pre><h2 id="field-comments">FIELD COMMENTS</h2>
<div class="cms-section">

<h3 id="locus"><a href="#LocusA">LOCUS</a></h3>
<p>The LOCUS field contains a number of different data elements, including locus name, sequence length, molecule type, GenBank division, and modification date. Each element is described below.</p>
<div class="cms-section">

<h3 id="locus-name">Locus Name</h3>
<p>The locus name in this example is <a href="#LocusA">SCU49845</a>.</p>
<p>The locus name was originally designed to help group entries with similar sequences: the first three characters usually designated the organism; the fourth and fifth characters were used to show other group designations, such as gene product; for segmented entries, the last character was one of a series of sequential integers. (See GenBank <a href="ftp://ftp.ncbi.nih.gov/genbank/gbrel.txt" target="_blank">release notes</a> section 3.4.4 for more info.)</p>
<p>However, the 10 characters in the locus name are no longer sufficient to represent the amount of information originally intended to be contained in the locus name. The only rule now applied in assigning a locus name is that it must be unique. For example, for GenBank records that have 6-character accessions (e.g., U12345), the locus name is usually the first letter of the genus and species names, followed by the accession number. For 8-character character accessions (e.g., AF123456), the locus name is just the accession number.</p>
<p>The <a href="refseq">RefSeq</a> database of reference sequences assigns formal locus names to each record, based on gene symbol. RefSeq is separate from the GenBank database, but contains cross-references to corresponding GenBank records.</p>
<p>Entrez Search Field: Accession Number [ACCN] Search Tip : It is better to search for the actual accession number rather than the locus name, because the accessions are stable and locus names can change.</p>
<div class="cms-section">

<h3 id="sequence-length"><a href="#SequenceLengthA">Sequence Length</a></h3>
<p>Number of nucleotide base pairs (or amino acid residues) in the sequence record. In this example, the sequence length is <a href="#SequenceLengthA">5028 bp</a>.</p>
<p>Entrez Search Field : Sequence Length [SLEN] Search Tips : (1) To retrieve records within a range of lengths, use the colon as the range operator, e.g., 2500:2600[SLEN]. (2) To retrieve all sequences shorter than a certain number, use 2 as the lower bound, e.g., 2:100[SLEN]. (3) To retrieve all sequences longer than a certain number, use a series of 9&apos;s as the upper bound, e.g., 325000:99999999[SLEN].</p>
<div class="cms-section">

<h3 id="molecule-type"><a href="#MoleculeTypeA">Molecule Type</a></h3>
<p>The type of molecule that was sequenced. In this example, the molecule type is <a href="#MoleculeTypeA">DNA</a>.</p>
<p>Each GenBank record must contain contiguous sequence data from a single molecule type. The various <a href="https://www.ncbi.nlm.nih.gov/Sequin//sequin.hlp.html#SpecifyMolecule" target="_blank">molecule types</a> are described in the Sequin documentation and can include genomic DNA, genomic RNA, precursor RNA, mRNA (cDNA), ribosomal RNA, transfer RNA, small nuclear RNA, and small cytoplasmic RNA.</p>
<p>Entrez Search Field : Properties [PROP] Search Tip : Search term should be in the format: biomol_genomic, biomol_mRNA, etc. For more examples, view the Properties field in the Index mode.</p>
<div class="cms-section">

<h3 id="genbank-division"><a href="#GenBankDivisionA">GenBank Division</a></h3>
<p>The GenBank division to which a record belongs is indicated with a three letter abbreviation. In this example, GenBank division is <a href="#GenBankDivisionA">PLN</a>.</p>
<p>The GenBank database is divided into 18 divisions:</p>
<ol>
<li>PRI - primate sequences</li>
<li>ROD - rodent sequences</li>
<li>MAM - other mammalian sequences</li>
<li>VRT - other vertebrate sequences</li>
<li>INV - invertebrate sequences</li>
<li>PLN - plant, fungal, and algal sequences</li>
<li>BCT - bacterial sequences</li>
<li>VRL - viral sequences</li>
<li>PHG - bacteriophage sequences</li>
<li>SYN - synthetic sequences</li>
<li>UNA - unannotated sequences</li>
<li>EST - EST sequences (expressed sequence tags)</li>
<li>PAT - patent sequences</li>
<li>STS - STS sequences (sequence tagged sites)</li>
<li>GSS - GSS sequences (genome survey sequences)</li>
<li>HTG - HTG sequences (high-throughput genomic sequences)</li>
<li>HTC - unfinished high-throughput cDNA sequencing</li>
<li>ENV - environmental sampling sequences</li>
</ol>
<p>Some of the divisions contain sequences from specific groups of organisms, whereas others (EST, GSS, HTG, etc.) contain data generated by specific sequencing technologies from many different organisms. The organismal divisions are historical and do not reflect the current<a href="taxonomy">NCBI Taxonomy</a>. Instead, they merely serve as a convenient way to divide GenBank into smaller pieces for those who want to FTP the database. Because of this, and because sequences from a particular organism can exist in technology-based divisions such as EST, HTG, etc., the<a href="taxonomy">NCBI Taxonomy Browser</a> should be used for retrieving all sequences from a particular organism.</p>
<p>Entrez Search Field : Properties [PROP] Search Tip : Search term should be in the format: gbdiv_pri, gbdiv_est, etc. For more examples, view the Properties field in the Index mode. For example, to eliminate all sequences from a particular division, such as all ESTs, you can use a Boolean query formatted such as: human[ORGN] NOT gbdiv_est[PROP] For the reasons noted above, do not use GenBank divisions to retrieve all sequences from a specific organism. Instead, use the <a href="Taxonomy/Browser/wwwtax.cgi">NCBI Taxonomy Browser</a>.</p>
<div class="cms-section">

<h3 id="modification-date"><a href="#ModificationDateA">Modification Date</a></h3>
<p>The date in the LOCUS field is the date of last modification. The sample record shown here was last modified on <a href="#ModificationDateA">21-JUN-1999</a>.</p>
<p>Entrez Search Field : Modification Date [MDAT] Search Tips : (1) Enter search term in the format: yyyy/mm/dd, e.g., 1999/07/25. (2) To retrieve records modified between two dates, use the colon as a range operator, e.g., 1999/07/25:1999/07/31[MDAT]. (3) You can use the Publication Date [PDAT] field of Entrez to limit search results by the date on which records were added to the Entrez system. Publication date can be in the form of a range, just like the Modification Date.</p>
<h3 id="definition"><a href="#DefinitionA">DEFINITION</a></h3>
<p>Brief description of sequence; includes information such as source organism, gene name/protein name, or some description of the sequence&apos;s function (if the sequence is non-coding). If the sequence has a coding region (CDS), description may be followed by a completeness qualifier, such as &quot;complete cds&quot;.</p>
<p>Entrez Search Field: Title Word [TITL] Search Tip : Although nucleotide definition lines follow a <a href="https://www.ncbi.nlm.nih.gov/Sequin/sequin.hlp.html#NucleotideDefinitionline" target="_blank">structured format</a>, GenBank does not use a controlled vocabulary, and authors determine the content of their records. Therefore, if a search for a specific term does not retrieve the desired records, try other terms that authors might have used, such as synonyms, full spellings, or abbreviations. The &quot;related records&quot; (or &quot;neighbors&quot;) function of Entrez also allows you to broaden your search by retrieving records with similar sequences, regardless of the descriptive terms used by the submitters.</p>
<div class="cms-section">

<h3 id="accession"><a href="#AccessionA">ACCESSION</a></h3>
<p>The unique identifier for a sequence record. An accession number applies to the complete record and is usually a combination of a letter(s) and numbers, such as a single letter followed by five digits (e.g., U12345) or two letters followed by six digits (e.g., AF123456). Some accessions might be longer, depending on the type of sequence record.</p>
<p>Accession numbers do not change, even if information in the record is changed at the author&apos;s request. Sometimes, however, an original accession number might become secondary to a newer accession number, if the authors make a new submission that combines previous sequences, or if for some reason a new submission supercedes an earlier record.</p>
<p>Records from the <a href="refseq">RefSeq</a> database of reference sequences have a their own <a href="www.ncbi.nlm.nih.gov/books/NBK21091/table/ch18.T.refseq_accession_numbers_and_mole">accession number format</a> that begins with two letters followed by an underscore bar and six or more digits; for example:</p>
<pre>NT_123456   constructed genomic contigs
NM_123456   mRNAs
NP_123456   proteins
NC_123456   chromosomes
</pre>

<p><strong>Note:</strong> Most records have both a series of accession numbers (<a href="#VersionB">Version</a> for nucleotide sequences and <a href="#ProteinIDB">protein_id</a> for amino acid sequences) and sequence identifiers (<a href="#GInB">GI</a> for nucleotide sequences and <a href="#GIpB">GI</a> for amino acid sequences). See the online documentation for <a href="~/sequenceids">Sequence IDs</a> for details.</p>
<p>Entrez Search Field: Accession [ACCN] Search Tip : The letters in the accession number can be written in upper- or lowercase. RefSeq accessions must contain an underscore bar between the letters and the numbers, e.g., NM_002111.</p>
<div class="cms-section">

<h3 id="version"><a href="#VersionA">VERSION</a></h3>
<p>A nucleotide sequence identification number that represents a single, specific sequence in the GenBank database. This identification number uses the accession.version format implemented by GenBank/EMBL/DDBJ in February 1999.</p>
<p>If there is any change to the sequence data (even a single base), the version number will be increased, e.g., U12345.1 ? U12345.2, but the accession portion will remain stable.</p>
<p>The accession.version system of sequence identifiers runs parallel to the <a href="#GInB">GI</a> number system--when any change is made to a sequence, it receives a new GI number AND its version number is incremented by one.</p>
<p>To find out about the revision history of a sequence, see <a href="~/sequencerevisionhistory">GenBank Sequence Revision History</a>.</p>
<p>Entrez Search Field: use the default setting of &quot;All Fields&quot;</p>
<div class="cms-section">

<h3 id="gi"><a href="#GInA">GI</a></h3>
<p>&quot;GenInfo Identifier&quot; sequence identification number, in this case, for the nucleotide sequence. If a sequence changes in any way, a new GI number will be assigned.</p>
<p>A separate GI number is also assigned to each protein translation within a nucleotide sequence record, and a new GI is assigned if the protein translation changes in any way (see <a href="#GIpB">below</a> ).</p>
<p>GI sequence identifiers run parallel to the new accession.version system of sequence identifiers.</p>
<p>Read more about <a href="~/sequencerevisionhistory">GenBank Sequence Revision History</a> and <a href="~/sequenceids">Sequence IDs</a>.</p>
<p>Entrez Search Field: use the default setting of &quot;All Fields&quot;</p>
<div class="cms-section">

<h3 id="keywords"><a href="#KeywordsA">KEYWORDS</a></h3>
<p>Word or phrase describing the sequence. If no keywords are included in the entry, the field contains only a period.</p>
<p>The Keywords field is present in sequence records primarily for historical reasons, and is not based on a controlled vocabulary. Keywords are generally present in older records. They are not included in newer records unless the record contains a special type of sequence such as EST, STS, GSS, HTG, etc.</p>
<p>Entrez Search Field: Keyword [KYWD] Search Tip : Because keywords are not present in many records, it is best not to search that field. Instead, search All Fields [ALL], the Text Word [WORD] field, or the Title Word [TITL] field, for progressively narrower retrieval.</p>
<div class="cms-section">

<h3 id="source"><a href="#SourceA">SOURCE</a></h3>
<p>Free-format information including an abbreviated form of the organism name, sometimes followed by a molecule type.</p>
<p>Entrez Search Field: Organism [ORGN] Search Tip : For some organisms that have well-established common names, such as baker&apos;s yeast, mouse, and human, a search for the common name will yield the same results as a search for the scientific name, e.g., a search for &quot;baker&apos;s yeast&quot; in the organism field retrieves the same number of documents as &quot;Saccharomyces cerevisiae&quot;. This is true because the Organism field is connected to the <a href="taxonomy">NCBI Taxonomy Database</a>, which contains cross-references between common names, scientific names, and synonyms for organisms represented in the Sequence databases.</p>
<div class="cms-section">

<h3 id="organism"><a href="#OrganismA">Organism</a></h3>
<p>The formal scientific name for the source organism (genus and species, where appropriate) and its lineage, based on the phylogenetic classification scheme used in the <a href="taxonomy">NCBI Taxonomy Database</a>. If the complete lineage of an organism is very long, an abbreviated lineage will be shown in the GenBank record and the complete lineage will be available in the Taxonomy Database. (See also the <a href="#TaxonB">/db_xref=taxon:nnnn</a> Feature qualifer, below.)</p>
<p>Entrez Search Field: Organism [ORGN] Search Tip : You can search the Organism field by any node in the taxonomic hierarchy, e.g., you can search for the term &quot;Saccharomyces cerevisiae&quot;, &quot;Saccharomycetales&quot;, &quot;Ascomycota&quot;, etc. to retrieve all the sequences from organisms in a particular taxon.</p>
<div class="cms-section">

<h3 id="reference"><a href="#ReferenceA">REFERENCE</a></h3>
<p>Publications by the authors of the sequence that discuss the data reported in the record. References are automatically sorted within the record based on date of publication, showing the oldest references first.</p>
<p>Some sequences have not been reported in papers and show a status of &quot;unpublished&quot; or &quot;in press&quot;. When an accession number and/or sequence data has appeared in print, sequence authors should send the complete citation of the article to update@ncbi.nlm.nih.gov and the GenBank staff will revise the record.</p>
<p>Various <a href="https://www.ncbi.nlm.nih.gov/Sequin/sequin.hlp.html#Class" target="_blank">classes</a> of publication can be present in the References field, including journal article, book chapter, book, thesis/monograph, proceedings chapter, proceedings from a meeting, and patent.</p>
<p>The last citation in the REFERENCE field usually contains information about the submitter of the sequence, rather than a literature citation. It is therefore called the &quot;submitter block&quot; and shows the words &quot;Direct Submission&quot; instead of an article title. Additional information is provided below, under the header <a href="#SubmitterBlockB">Direct Submission</a>. Some older records do not contain a submitter block.</p>
<p>Entrez Search Field: The various subfields under References are searchable in the Entrez search fields noted below.</p>
<div class="cms-section">

<h3 id="authors"><a href="#AuthorsA">AUTHORS</a></h3>
<p>List of authors in the order in which they appear in the cited article.</p>
<p>Entrez Search Field: Author [AUTH] Search Tip : Enter author names in the form: Lastname AB (without periods after the initials). Initials can be omitted. Truncation can also be used to retrieve all names that begin with a character string, e.g., Richards<em> or Boguski M</em>.</p>
<div class="cms-section">

<h3 id="title"><a href="#TitleA">TITLE</a></h3>
<p>Title of the published work or tentative title of an unpublished work.</p>
<p>Sometimes the words &quot;Direct Submission&quot; instead of an article title. This is usually true for the last citation in the <a href="#ReferenceB">REFERENCE</a> field because it tends to contain information about the submitter of the sequence, rather than a literature citation. The last citation is therefore called the &quot;submitter block&quot;. Additional information is provided below, under the header <a href="#SubmitterBlockB">Direct Submission</a>. Some older records do not contain a submitter block.</p>
<p>Entrez Search Field: Text Word [WORD] Note: For sequence records, the Title Word [TITL] field of Entrez searches the <a href="#DefinitionB">Definition Line</a>, not the titles of references listed in the record. Therefore, use the Text Word field to search the titles of references (and other text-containing fields). Search Tip : If a search for a specific term does not retrieve the desired records, try other terms that authors might have used, such synonyms, full spellings, or abbreviations. The &apos;related records&apos; (or &apos;neighbors&apos;) function of Entrez also allows you to broaden your search by retrieving records with similar sequences, regardless of the descriptive terms used by the submitters.</p>
<div class="cms-section">

<h3 id="journal"><a href="#JournalA">JOURNAL</a></h3>
<p>MEDLINE abbreviation of the journal name. (Full spellings can be obtained from the Entrez <a href="journals">Journals Database</a>.)</p>
<p>Entrez Search Field: Journal Name [JOUR] Search Tip : Journal names can be entered as either the full spelling or the MEDLINE abbreviation. You can search the Journal Name field in the Index mode to see the index for that field, and to select one or more journal names for inclusion in your search.</p>
<div class="cms-section">

<h3 id="pubmed"><a href="#PubMedA">PUBMED</a></h3>
<p>PubMed Identifier (PMID).</p>
<p>References that include PubMed IDs contain links from the sequence record to the corresponding PubMed record. Conversely, PubMed records that contain accession number(s) in the SI (secondary source identifier) field contain links back to the sequence record(s).</p>
<p>Entrez Search Field: It is not possible to search the Nucleotide or Protein sequence databases by PubMed ID. However, you can search the PubMed (literature) database of Entrez for the PubMed ID, and then link to the associated sequence records.</p>
<div class="cms-section">

<h3 id="direct-submission"><a href="#SubmitterBlockA">Direct Submission</a></h3>
<p>Contact information of the submitter, such as institute/department and postal address. This is always the last citation in the References field. Some older records do not contain the &quot;Direct Submission&quot; reference. However, it is required in all new records.</p>
<p>The Authors subfield contains the submitter name(s), Title contains the words &quot;Direct Submission&quot;, and Journal contains the address.</p>
<p>The date in the Journal subfield is the date on which the author prepared the submission. In many cases, it is also the date on which the sequence was received by the GenBank staff, but it is not the date of first public release.</p>
<p>Entrez Search Field: Use the Author Field [AUTH] if searching for the author name. Use All Fields [ALL] if searching for an element of the author&apos;s address (e.g., Yale University). Note, however, that retrieved records might contain the institution name in a field such as Comment, rather than in the Direct Submission reference, so you might get some false hits. Search Tip : It is sometimes helpful to search for both the full spelling and an abbreviation, e.g., &quot;Washington University&quot; OR &quot;WashU&quot;, because the spelling used by authors might vary.</p>
<div class="cms-section">

<h3 id="features"><a href="#FeaturesA">FEATURES</a></h3>
<p>Information about genes and gene products, as well as regions of biological significance reported in the sequence. These can include regions of the sequence that code for proteins and RNA molecules, as well as a number of other features.</p>
<p>A complete list of features is available in the following places:</p>
<ul>
<li><a href="www.insdc.org/documents/feature_table.html#7.3">Appendix III: Feature keys reference</a> of the <a href="www.insdc.org/documents/feature_table.html">DDBJ/EMBL/GenBank Feature Table</a> provides definitions, optional qualifiers, and comments for each feature. An <a href="www.insdc.org/documents/feature_table.html#7.3.2">alphabetical list</a> is also available. <a href="www.insdc.org/documents/feature_table.html#7.4">Appendix IV: Summary of qualifiers for feature keys</a> provides definitions for the Feature qualifiers.</li>
<li><a href="https://www.ncbi.nlm.nih.gov/Sequin/sequin.hlp.html" target="_blank">Sequin Help documentation</a> (scroll down to &apos;Features&apos; in the table of contents to see an alphabetical list of features with links to descriptions)</li>
</ul>
<p>The location of each feature is provided as well, and can be a single base, a contiguous span of bases, a joining of sequence spans, and other representations. If a feature is located on the complementary strand, the word &quot;<a href="#ComplementA">complement</a>&quot; will appear before the base span. If the &quot;<code>&lt;</code>&quot; symbol precedes a base span, the sequence is partial on the 5&apos; end (e.g., <code>CDS &lt;1..206</code>). If the &quot;<code>&gt;</code>&quot; symbol follows a base span, the sequence is partial on the 3&apos; end (e.g., <code>CDS 435..915&gt;</code>).</p>
<p>For more information about feature locations, see the <a href="https://www.ncbi.nlm.nih.gov/Sequin/sequin.hlp.html#Locationpage" target="_blank">Sequin Help Documentation</a>.</p>
<p>The sample record shown here only includes a small number of features (source, CDS, and gene, all of which are described below). The <a href="#OtherFeaturesB">Other Features</a> section, below, provides links to some GenBank records that show a variety of additional features.</p>
<p>Entrez Search Field: Feature Key [FKEY] Search Tip : To scroll through the list of available features, view the Feature Key field in Index mode. You can then select one or more features from the index to include in your query. For example, you can limit your search to records that contain both primer_bind and promoter features.</p>
<div class="cms-section">

<h3 id="source"><a href="#FeaturesSourceA">source</a></h3>
<p>Mandatory feature in each record that summarizes the length of the sequence, scientific name of the source organism, and Taxon ID number. Can also include other information such as map location, strain, clone, tissue type, etc., if provided by submitter.</p>
<p>Entrez Search Field: All Fields [ALL] can be used to search for some elements in the source field, such as strain, clone, tissue type.</p>
<p>Use the <a href="#SequenceLengthB">Sequence Length</a> [SLEN] field to search by length and the <a href="#OrganismB">Organism</a> [ORGN] field to search by organism name.</p>
<p>Because map location is written as free text and can be represented in a number of ways (e.g., chromosome number, cytogenetic location, marker name, physical map location), it is not directly searchable in the Entrez Nucleotide or Protein databases. However, there are a number of resources that allow you to browse and/or search the <a href="guide/genomes-maps">maps of various genomes</a>.</p>
<div class="cms-section">

<h3 id="taxon"><a href="#TaxonA">Taxon</a></h3>
<p>A stable unique identification number for the taxon of the source oganism. A taxonomy ID number is assigned to each taxon (species, genus, family, etc.) in the <a href="taxonomy">NCBI Taxonomy Database</a>. See also the <a href="#OrganismB">Organism</a> field, above.</p>
<p>Entrez Search Field: The Taxonomy ID number is not searchable in the Organism search field of Entrez but is searchable in the <a href="taxonomy">Taxonomy Browser</a>.</p>
<p>Note: The <a href="#TaxonA">/db_xref qualifier</a> is one of many that can be applied to various features. A complete list is available in <a href="www.insdc.org/documents/feature_table.html#7.4">Appendix IV: Summary of qualifiers for feature keys</a> of the <a href="www.insdc.org/documents/feature_table.html">DDBJ/EMBL/GenBank Feature Table</a>, and in section 3.4.12.3 of the GenBank <a href="ftp://ftp.ncbi.nih.gov/genbank/gbrel.txt" target="_blank">release notes</a>. <a href="projects/collab/FT/index#7.3">Appendix III: Feature keys reference</a> shows which qualifiers can be used with specific features (see <a href="www.insdc.org/documents/feature_table.html#7.3.2">alphabetical list</a> ).</p>
<div class="cms-section">

<h3 id="cds"><a href="#CDSA">CDS</a></h3>
<p>Coding sequence; region of nucleotides that corresponds with the sequence of amino acids in a protein (location includes start and stop codons). The CDS feature includes an amino acid <a href="#TranslationB">translation</a>. Authors can specify the nature of the CDS by using the qualifier &quot;/evidence=experimental&quot; or &quot;/evidence=not_experimental&quot;.</p>
<p>Submitters are also encouraged to annotate the mRNA feature, which includes the 5&apos; untranslated region (5&apos;UTR), coding sequences (CDS, exon), and 3&apos; untranslated region (3&apos;UTR).</p>
<p>Entrez Search Field: Feature Key [FKEY] Search Tip : You can use this field to limit your search to records that contain a particular feature, such as CDS. To scroll through the list of available features, view the Feature Key field in Index mode. A complete list of features is also available from the resources noted <a href="#FeaturesB">above</a>.</p>
<div class="cms-section">

<h3 id="1206"><a href="#BaseSpanA">&lt;1..206</a></h3>
<p>Base span of the biological feature indicated to the left, in this case, a CDS feature. (The CDS feature is described <a href="#CDSB">above</a>, and its base span includes the start and stop codons.) Features can be complete, partial on the 5&apos; end, partial on the 3&apos; end, and/or on the complementary strand. Examples:</p>
<ol>
<li>A complete feature is simply written as n..m. Example: <code>687..3158</code>The feature extends from base 687 through base 3158 in the sequence shown</li>
<li>The <code>&lt;</code>symbol indicates partial on the 5&apos; end. Example: <code>&lt;1..206</code>. The feature extends from base 1 through base 206 in the sequence shown, and is partial on the 5&apos; end</li>
<li>The <code>&gt;</code>symbol indicates partial on the 3&apos; end. Example: <code>4821..5028&gt;</code>. The feature extends from base 4821 through base 5028 and is partial on the 3&apos; end</li>
<li><code>complement(range)</code>indicates that the feature is on the complementary strand. Example: <code>complement(3300..4037)</code>. The feature extends from base 3300 through base 4037 but is actually on the complementary strand. It is therefore read in the opposite direction on the reverse complement sequence. (For an example, see <a href="#ComplementA">the third CDS feature</a> in the sample record shown on this page. In this case, the amino acid translation is generated by taking the reverse complement of bases 3300 to 4037 and reading that reverse complement sequence in its 5&apos; to 3&apos; direction.)</li>
</ol>
<div class="cms-section">

<h3 id="proteinid"><a href="#ProteinIDA">protein_id</a></h3>
<p>A protein sequence identification number, similar to the <a href="#VersionB">Version</a> number of a nucleotide sequence. Protein IDs consist of three letters followed by five digits, a dot, and a version number. If there is any change to the sequence data (even a single amino acid), the version number will be increased, but the accession portion will remain stable (e.g., AAA98665.1 will change to AAA98665.2).</p>
<p>The accession.version format of protein sequence identification numbers was implemented by GenBank/EMBL/DDBJ in February 1999 and runs parallel to the GI number system. More details about sequence identification numbers and the difference between GI number and version are provided in <a href="~/sequenceids">Sequence Identifiers: A Historical Note</a>.</p>
<p>Entrez Search Field: use the default setting of &quot;All Fields&quot;</p>
<div class="cms-section">

<h3 id="gi"><a href="#GIpA">GI</a></h3>
<p>&quot;GenInfo Identifier&quot; sequence identification number, in this case, for the protein translation.</p>
<p>The GI system of sequence identifiers runs parallel to the accession.version system, which was implemented by GenBank, EMBL, and DDBJ in February 1999. Therefore, if the protein sequence changes in any way, it will receive a new GI number, and the suffix of the <a href="#ProteinIDB">protein_id</a> will be incremented by one..</p>
<p>More details about sequence identification numbers and the difference between GI number and version are provided in <a href="~/sequenceids">Sequence IDs</a>.</p>
<p>Entrez Search Field: use the default setting of &quot;All Fields&quot;</p>
<div class="cms-section">

<h3 id="translation"><a href="#TranslationA">translation</a></h3>
<p>The amino acid translation corresponding to the nucleotide coding sequence ( <a href="#CDSB">CDS</a> ). In many cases, the translations are conceptual. Note that authors can indicate whether the CDS is based on experimental or non-experimental evidence.</p>
<p>Entrez Search Field: It is not possible to search the translation subfield using Entrez. If you want use a string of amino acids as a query to retrieve similar protein sequences, use <a href="blast">BLAST</a> instead.</p>
<div class="cms-section">

<h3 id="gene"><a href="#GeneA">gene</a></h3>
<p>A region of biological interest identified as a gene and for which a name has been assigned. The base span for the gene feature is dependent on the furthest 5&apos; and 3&apos; features. Additional examples of records that show the relationship between gene features and other features such as mRNA and CDS are <a href="nuccore/AF165912">AF165912</a> and <a href="nuccore/AF090832">AF090832</a>.</p>
<p>Entrez Search Field: Feature Key [FKEY] Search Tip : You can use this field to limit your search to records that contain a particular feature, such as a gene. To scroll through the list of available features, view the Feature Key field in Index mode. A complete list of features is also available from the resources noted <a href="#FeaturesB">above</a>.</p>
<div class="cms-section">

<h3 id="complement"><a href="#ComplementA">complement</a></h3>
<p>Indicates that the feature is located on the complementary strand.</p>
<div class="cms-section">

<h3 id="other-features">Other Features</h3>
<p>Examples of other records that show a variety of biological features; a graphic format is also available for each sequence record and visually represents the annotated features:</p>
<ul>
<li><a href="nucleotide/AF165912">AF165912 (gene, promoter, TATA signal, mRNA, 5&apos;UTR, CDS, 3&apos;UTR)</a></li>
<li><a href="nucleotide/AF090832">AF090832</a> (protein bind, gene, 5&apos;UTR, mRNA, CDS, 3&apos;UTR)</li>
<li><a href="nucleotide/L00727">L00727</a> (alternatively spliced mRNAs)</li>
</ul>
<p>A complete list of features is available from the resources noted <a href="#FeaturesB">above</a>.</p>
<div class="cms-section">

<h3 id="origin"><a href="#OriginA">ORIGIN</a></h3>
<p>The ORIGIN may be left blank, may appear as &quot;Unreported,&quot; or may give a local pointer to the sequence start, usually involving an experimentally determined restriction cleavage site or the genetic locus (if available). This information is present only in older records.</p>
<p>The sequence data begin on the line immediately below ORIGIN. To view or download the sequence data in <a href="blast/blastcgihelp.shtml">FASTA format</a>, append <code>?format=fasta</code>to the record&apos;s URL; for example, <a href="nucleotide/U49845">/nucleotide/U49845?format=fasta&amp;report=text</a>.</p>
</div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div>
          

        </div>
      </div>
    

    <div id='shared-content-1' nid='1092'>
      <div class='rightnav'>
        <h2>GenBank Resources</h2>
        <ul>
          <li>
            <a href="/~/">GenBank Home</a></li>
          <li>
            <a href="/~/submit_types">Submission Types</a></li>
          <li>
            <a href="/~/submit">Submission Tools</a></li>
          <li>
            <a href="http://www.ncbi.nlm.nih.gov/nuccore/">Search GenBank</a></li>
          <li>
            <a href="/~/update">Update GenBank Records</a></li>
        </ul>
      </div>
    </div>

  </body>
</html>

